<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">
<head>
<meta charset="UTF-8"/>
<title>List of update sites - ImageJ</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"List_of_update_sites","wgTitle":"List of update sites","wgCurRevisionId":38812,"wgRevisionId":38812,"wgArticleId":1690,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":[],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"List_of_update_sites","wgRelevantArticleId":1690,"wgRequestId":"a4912561488a08ed90e2f539","wgIsProbablyEditable":false,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgPreferredVariant":"en","fancytree_path":"/extensions/TreeAndMenu/fancytree"});mw.loader.implement("user.options",function($,jQuery,require,module){mw.user.options.set({"variant":"en"});});mw.loader.implement("user.tokens",function ( $, jQuery, require, module ) {
mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});/*@nomin*/;

});mw.loader.load(["mediawiki.page.startup"]);});</script>
<link rel="stylesheet" href="common.css"/>
<link rel="stylesheet" href="fancytree.css"/><link rel="stylesheet" href="suckerfish.css"/>
<meta name="ResourceLoaderDynamicStyles" content=""/>
<link rel="stylesheet" href="style.css"/>
<script async="" src="common.js"></script>
<meta name="generator" content="https://github.com/imagej/list-of-update-sites"/>
<meta name="description" content="The list of built-in update sites for ImageJ."/>
<link rel="shortcut icon" href="ij2.ico"/>
	<meta property="og:type" content="article"/>

	<meta property="og:site_name" content="ImageJ"/>

	<meta property="og:title" content="List of update sites"/>

	<meta property="og:description" content="A human-readable table of ImageJ's built-in list of update sites."/>

	<meta property="og:url" content="https://imagej.net/List_of_update_sites"/>

<meta name="viewport" content="width=device-width, initial-scale=1" />
</head>
<body class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-List_of_update_sites rootpage-List_of_update_sites skin-erudite action-view">

		<div class="mw-jump">
			<a href="#bodyContent">Skip to content</a>, 			<a href="#search">Skip to search</a>
		</div>

		<div id="top-wrap" role="banner">
			<h1><a href="https://imagej.net/Welcome" title="ImageJ" rel="home">ImageJ</a></h1>
			<div id="tagline">From ImageJ</div>

			<a id="menubutton" href="#menu">Menu</a>
			<div id="nav" role="navigation">
			<ul id='menu'>
<li id="menu-item-n-About"><a href="https://imagej.net/ImageJ">About</a></li>
<li id="menu-item-n-Downloads"><a href="https://imagej.net/Downloads">Downloads</a></li>
<li id="menu-item-n-Learn"><a href="https://imagej.net/Learn">Learn</a></li>
<li id="menu-item-n-Develop"><a href="https://imagej.net/Development">Develop</a></li>
<li id="menu-item-n-News"><a href="https://imagej.net/News">News</a></li>
<li id="menu-item-n-Events"><a href="https://imagej.net/Events">Events</a></li>
<li id="menu-item-n-Status"><a href="https://imagej.net/Status">Status</a></li>
<li id="menu-item-n-Help"><a href="https://imagej.net/Help">Help</a></li>
</ul>
			</div>
		</div>

		<div id="mw-js-message"></div>
		
		<div id="main" role="main">
			<div id="nav-meta">
			<span id="ca-nstab-main" class="selected"><a href="https://github.com/imagej/list-of-update-sites" title="Edit this list" accesskey="f">Edit this list</a></span></div>

			<div id="bodyContent">
				<h1>List of update sites</h1>
				
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div style="overflow:hidden"><table class="metadata plainlinks ambox ambox-content">
<tr>
<td class="ambox-image">
<p><a href="https://imagej.net/File:Important-sign.png" class="image"><img alt="Important-sign.png" src="https://imagej.net/_images/thumb/f/f0/Important-sign.png/40px-Important-sign.png" width="40" height="40" srcset="https://imagej.net/_images/thumb/f/f0/Important-sign.png/60px-Important-sign.png 1.5x, https://imagej.net/_images/thumb/f/f0/Important-sign.png/80px-Important-sign.png 2x" /></a>
</p>
</td>
<td> Update sites are a fantastic way to get new functionality in your ImageJ app.<br />Unfortunately they are also one of the easiest ways to break an installation, by adding an update site that clashes with another update site, or even the ImageJ core.<br />So, before enabling a new update site, it's safest to <a href="https://imagej.net/Make_Fiji_Package" title="Make Fiji Package">back up your installation</a>. </td>
<p><br />
</p>
</tr>
</table></div>
<table class="infobox" cellspacing="5" style="text-align: left; float: right; border: 1px solid #a0a0a0; font-size: 80%;">
<tr>
<th colspan="2" style="font-size: 130%;">  Update Sites
</th></tr>
<tr>
<td>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/Update_Sites" title="Update Sites">Introduction</a>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/Following_an_update_site" title="Following an update site">Following an update site</a>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/How_to_set_up_and_populate_an_update_site" title="How to set up and populate an update site">Creating your own update site</a>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/Personal_Update_Site_Terms_of_Service" title="Personal Update Site Terms of Service">Terms of Service for personal update  sites</a>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/Automatic_Update_Site_Uploads" title="Automatic Update Site Uploads">Automatically upload your site</a>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/How_to_upload_to_core_update_sites" title="How to upload to core update sites">Uploading to core update sites</a>
</td></tr>
<tr>
<td>  <strong class="selflink">List of update sites</strong>
</td></tr>
<tr>
<td>  <a href="https://imagej.net/Update_site_FAQ" title="Update site FAQ">Update site FAQ</a>
</td></tr></table>
<h2><span class="mw-headline" id="Introduction">Introduction</span></h2>
<p>The <a href="https://imagej.net/Updater" title="Updater">Updater</a> has a convenient <i>Manage update sites</i> dialog that is populated by a central index.
</p>
<ul><li> <b>Users:</b> You can <a href="https://imagej.net/How_to_follow_a_3rd_party_update_site" title="How to follow a 3rd party update site" class="mw-redirect">enable an update site</a> using the <span><em><span style="border-bottom:1px dotted #ccc;"> Help </span>&#160;&#9654; <span style="border-bottom:1px dotted #ccc;"> Update...</span></em></span> command and clicking the "Manage update sites" button.</li>
<li> <b>Developers:</b> You can add <a href="https://imagej.net/How_to_set_up_and_populate_an_update_site" title="How to set up and populate an update site">your own update site</a> to this list by filing a <a href="https://help.github.com/articles/about-pull-requests/">pull request</a> against <a href="https://github.com/imagej/list-of-update-sites">this repository</a> adding your site to the <code>sites.yml</code> index. Please maintain alphabetical order.</li></ul>
<h2><span class="mw-headline" id="Available_update_sites">Available update sites</span></h2>
<table class="wikitable">
<tr>
<th colspan="4" style="border-bottom: 1px solid gray">ImageJ Update Sites
</th></tr>
<tr>
<td style="vertical-align: bottom;"><b>Name</b>
</td>
<td style="vertical-align: bottom;"><b>Site</b>
</td>
<td style="vertical-align: bottom;"><b>Description</b>
</td>
<td style="vertical-align: bottom;"><b>Maintainer</b>
</td></tr>

<tr valign="top">
<td><b>ImageJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://update.imagej.net/">https://update.imagej.net/</a>
</td>
<td>Base update site for <a href="https://imagej.net/software/imagej2">ImageJ2</a>. No need to add this to your list of update sites; it is already included.
</td>
<td><a href="https://imagej.net/people#imagej2">ImageJ2 developers</a>
</td></tr>

<tr valign="top">
<td><b>Fiji</b>
</td>
<td><a rel="nofollow" class="external free" href="https://update.fiji.sc/">https://update.fiji.sc/</a>
</td>
<td>Base update site for <a href="https://imagej.net/software/fiji">Fiji</a> and <a href="https://imagej.net/list-of-extensions">core Fiji plugins</a>. If you downloaded Fiji directly, it is already included.
</td>
<td><a href="https://imagej.net/people#fiji">Fiji developers</a>
</td></tr>

<tr valign="top">
<td><b>ImageJ (Europe mirror)</b>
</td>
<td><a rel="nofollow" class="external free" href="https://downloads.micron.ox.ac.uk/fiji_update/mirrors/imagej/">https://downloads.micron.ox.ac.uk/fiji_update/mirrors/imagej/</a>
</td>
<td>European mirror for the ImageJ update site.  Replace the ImageJ update site with this one for faster updates in Europe.
</td>
<td><a href="https://imagej.net/people#imagej2">ImageJ2 developers</a>
</td></tr>

<tr valign="top">
<td><b>Fiji (Europe mirror)</b>
</td>
<td><a rel="nofollow" class="external free" href="https://downloads.micron.ox.ac.uk/fiji_update/mirrors/fiji/">https://downloads.micron.ox.ac.uk/fiji_update/mirrors/fiji/</a>
</td>
<td>European mirror for the Fiji update site.  Replace the Fiji update site with this one for faster updates in Europe.
</td>
<td><a href="https://imagej.net/people#fiji">Fiji developers</a>
</td></tr>

<tr valign="top">
<td><b>Fiji-Legacy</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Fiji-Legacy/">https://sites.imagej.net/Fiji-Legacy/</a>
</td>
<td>Restores old Fiji plugins which never made the transition to Maven, as well as obsolete libraries no longer used by current Fiji installations.
</td>
<td>UNSUPPORTED
</td></tr>

<tr valign="top">
<td><b>Java-8</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Java-8/">https://sites.imagej.net/Java-8/</a>
</td>
<td>Temporary base update site for <a href="https://imagej.net/software/imagej2">ImageJ2</a> and <a href="https://imagej.net/software/fiji">Fiji</a> components as everything migrates to Java 8. <strong>Will not work with Java 6 or 7.</strong> See <a href="https://imagej.net/news/2016-05-10-imagej-howto-java-8-java-6-java-3d">this page</a> for details.
</td>
<td><a href="https://imagej.net/people">ImageJ2 and Fiji developers</a>
</td></tr>

<tr valign="top">
<td><b>Java-8 (Europe mirror)</b>
</td>
<td><a rel="nofollow" class="external free" href="https://downloads.micron.ox.ac.uk/fiji_update/mirrors/java-8/">https://downloads.micron.ox.ac.uk/fiji_update/mirrors/java-8/</a>
</td>
<td>European mirror for the Java-8 update site.  Replace the Java-8 update site with this one for faster updates in Europe.
</td>
<td><a href="https://imagej.net/people">ImageJ2 and Fiji developers</a>
</td></tr>

<tr valign="top">
<td><b>2015-Conference</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/2015-Conference/">https://sites.imagej.net/2015-Conference/</a>
</td>
<td>A collection of plugins highlighting the latest features of ImageJ. As seen in <a href="https://imagej.net/people/rasband">Wayne Rasband</a>'s <a href="https://vimeo.com/140929691">opening talk</a> at the <a href="https://imagej.net/events/conference-2015">2015 conference</a>!
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>3D ImageJ Suite</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Tboudier/">https://sites.imagej.net/Tboudier/</a>
</td>
<td>A <a href="https://imagejdocu.tudor.lu/doku.php?id=plugin:stacks:3d_ij_suite:start">suite of plugins</a> providing enhanced 3D capabilities in ImageJ.
</td>
<td><a href="https://imagej.net/people/mcib3d">Thomas Boudier</a>
</td></tr>

<tr valign="top">
<td><b>3Dscript</b>
</td>
<td><a rel="nofollow" class="external free" href="https://romulus.oice.uni-erlangen.de/updatesite/">https://romulus.oice.uni-erlangen.de/updatesite/</a>
</td>
<td><a href="https://imagej.net/plugins/3dscript">3Dscript</a> is a plugin animating 3D/4D microscopy data using a natural-language-based syntax.
</td>
<td><a href="https://imagej.net/people/bene51">Benjamin Schmid</a>
</td></tr>

<tr valign="top">
<td><b>ActogramJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://romulus.oice.uni-erlangen.de/imagej/updatesites/ActogramJ/">https://romulus.oice.uni-erlangen.de/imagej/updatesites/ActogramJ/</a>
</td>
<td><a href="https://bene51.github.io/ActogramJ">ActogramJ</a> is a plugin for analyzing actograms.
</td>
<td><a href="https://imagej.net/people/bene51">Benjamin Schmid</a>
</td></tr>

<tr valign="top">
<td><b>AdipoQ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/AdipoQ/">https://sites.imagej.net/AdipoQ/</a>
</td>
<td>ImageJ plugins for quantifying adipocyte morphology and function in tissues and in vitro. For more details, a link to the corresponding publication, and information on how to cite the plugins visit the AdipoQ github page https://github.com/hansenjn/AdipoQ/
</td>
<td><a href="https://github.com/hansenjn/">Jan Hansen</a>
</td></tr>

<tr valign="top">
<td><b>AIC Janelia - Course</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/AICjanelia-course/">https://sites.imagej.net/AICjanelia-course/</a>
</td>
<td>A collection of plugins and macros for students of the Advanced Imaging Center (AIC) Fiji Course.
</td>
<td><a href="https://imagej.net/orgs/janelia#advanced-imaging-center">AIC Janelia</a>
</td></tr>

<tr valign="top">
<td><b>ALPACA</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ALPACA/">https://sites.imagej.net/ALPACA/</a>
</td>
<td>A collection of macros to study cilia.
</td>
<td><a href="https://github.com/tschmenger/">Torsten Schmenger</a>
</td></tr>

<tr valign="top">
<td><b>AMPT</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/AMPT/">https://sites.imagej.net/AMPT/</a>
</td>
<td>The <a href="https://allenmli.github.io/AMPT/">Aquatic Mammal Photogrammetry Tool</a> (AMPT) is an ImageJ plug-in that simplifies the process of measuring aquatic mammals from aerial imagery. The tool may be configured to support different guided measurements.
</td>
<td><a href="mailto:ampt-dev@allenai.org">AMPT Developers</a>
</td></tr>

<tr valign="top">
<td><b>Angiogenesis</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Angiogenesis/">https://sites.imagej.net/Angiogenesis/</a>
</td>
<td>A plugin to quantify angiogenic sprout morphology.
</td>
<td><a href="https://imagej.net/people/imagejan">Jan Eglinger</a>
</td></tr>

<tr valign="top">
<td><b>AngioTool</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/AngioTool/">https://sites.imagej.net/AngioTool/</a>
</td>
<td>The <a href="https://ccrod.cancer.gov/confluence/display/ROB2/Home">AngioTool</a> software converted to a Fiji plugin.
</td>
<td><a href="https://imagej.net/people/imagejan">Jan Eglinger</a>
</td></tr>

<tr valign="top">
<td><b>apoc</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/apoc/">https://sites.imagej.net/apoc/</a>
</td>
<td>Accelerated pixel and object classifiers (APOC). See <a href="https://github.com/clij/clijx-accelerated-pixel-and-object-classification">clijx-accelerated-pixel-and-object-classification</a>.
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>Archipelago</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Lindsey/">https://sites.imagej.net/Lindsey/</a>
</td>
<td>A set of plugins that allow Fiji to be run in parallel on a cluster. Currently, <a href="https://imagej.net/plugins/tws">Trainable Weka Segmentation</a> and <a href="https://imagej.net/plugins/elastic-alignment-and-montage">Elastic Alignment</a> are accelerated in this way.
</td>
<td><a href="https://imagej.net/people/larrylindsey">Larry Lindsey</a>
</td></tr>

<tr valign="top">
<td><b>AxoNet</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/AxoNet/">https://sites.imagej.net/AxoNet/</a>
</td>
<td>An implementation of the AxoNet tool which can be used to count axons in light micrographs of optic nerves.
</td>
<td><a href="https://imagej.net/User:Mritch3">Matthew Ritch</a>
</td></tr>

<tr valign="top">
<td><b>BACMMAN</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BACMMAN/">https://sites.imagej.net/BACMMAN/</a>
</td>
<td>BACteria in Mother Machine ANalyzer: a plugin for high-throughput analysis of multichannel mother machine data. See <a href="https://github.com/jeanollion/bacmman">Sources</a>: https://github.com/jeanollion/bacmman and <a href="https://github.com/jeanollion/bacmman/wiki">Wiki</a>: https://github.com/jeanollion/bacmman/wiki
</td>
<td><a href="https://www.sabilab.fr/">Jean Ollion</a>
</td></tr>

<tr valign="top">
<td><b>BACMMAN-DL</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BACMMAN-DL/">https://sites.imagej.net/BACMMAN-DL/</a>
</td>
<td>Enables deep learning for BACMMAN (CPU). Currently using Tensorflow 2x
</td>
<td><a href="https://www.sabilab.fr/">Jean Ollion</a>
</td></tr>

<tr valign="top">
<td><b>BACMMAN-DL-GPU</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BACMMAN-DL-GPU/">https://sites.imagej.net/BACMMAN-DL-GPU/</a>
</td>
<td>Enables deep learning for BACMMAN (GPU). Currently using Tensorflow 2x
</td>
<td><a href="https://www.sabilab.fr/">Jean Ollion</a>
</td></tr>

<tr valign="top">
<td><b>BAR</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Tiago/">https://sites.imagej.net/Tiago/</a>
</td>
<td><a href="https://imagej.net/plugins/bar">BAR</a> (<em>Broadly Applicable Routines</em>) is a <a href="https://github.com/tferr/Scripts">GitHub-curated</a> collection of scripts that simplifies the daily usage of ImageJ. The collection is focused on Data Analysis, Image Annotation and Segmentation.
</td>
<td><a href="https://imagej.net/people/tferr">Tiago Ferreira</a>
</td></tr>

<tr valign="top">
<td><b>BaSiC</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BaSiC/">https://sites.imagej.net/BaSiC/</a>
</td>
<td><a href="https://www.helmholtz-muenchen.de/icb/research/groups/quantitative-single-cell-dynamics/software/basic/index.html">BaSiC</a> is a retrospective image correction method for optical microscopy, available as a Fiji/ImageJ Plugin. It is based on low rank and sparse decomposition, which corrects both shading (uneven illumination) in space and background bleaching in time.
</td>
<td><a href="mailto:tingying.peng@tum.de">Tingying Peng</a>
</td></tr>

<tr valign="top">
<td><b>Batch Composite</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BatchComposite/">https://sites.imagej.net/BatchComposite/</a>
</td>
<td><a href="https://imagej.net/plugins/batch-composite">Batch Composite</a> is a small plugin that runs the Composite operation on a folder of openable files. It supports Bio-Formats, recursion, and color customization.
</td>
<td><a href="mailto:kfish610@gmail.com">Kevin Fisher</a>
</td></tr>

<tr valign="top">
<td><b>Batch Tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BatchTools/">https://sites.imagej.net/BatchTools/</a>
</td>
<td><a href="https://imagej.net/plugins/batch-tools">Batch Tools</a> provides a toolbar to support batch image processing and analysis, including batch extract, merge channels, composite conversion, Z-projection, channel B&amp;C and colour adjustment, file export, and cell count, gray value &amp; area measurement.
</td>
<td><a href="https://github.com/neville-s-ng">Neville S Ng</a>
</td></tr>

<tr valign="top">
<td><b>BD-CellView</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BD-CellView/">https://sites.imagej.net/BD-CellView/</a>
</td>
<td><a href="https://github.com/embl-cba/ICS">BD-CellView</a> enables inspection and batch processing of images generated with the BD-CellView imaging flow cytometer.
</td>
<td><a href="mailto:christian.tischer@embl.de">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>BIG-EPFL</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BIG-EPFL/">https://sites.imagej.net/BIG-EPFL/</a>
</td>
<td>ImageJ plugins from <a href="https://imagej.net/licensing/big">EPFL's Biomedical Imaging Group (BIG)</a>. NOT OPEN SOURCE.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>BigDataProcessor</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BigDataProcessor/">https://sites.imagej.net/BigDataProcessor/</a>
</td>
<td><a href="https://github.com/embl-cba/bigDataProcessor2#bigdataprocessor2">BigDataProcessor2</a> is an Imglib2 and BigDataViewer based tool for inspection and processing of big (TB-sized) image data.
</td>
<td><a href="mailto:christian.tischer@embl.de">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>BigStitcher</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BigStitcher/">https://sites.imagej.net/BigStitcher/</a>
</td>
<td>Based on the HDF5 multi-resolution datastructures of the BigDataViewer the BigStitcher uses globally optimal registration that is capable of aligning large multi-tile datasets with high precision and automatic error detection within just a few minutes while at the same time allowing for additional user interaction.
</td>
<td><a href="https://imagej.net/people/StephanPreibisch">Stephan Preibisch</a>, <a href="https://imagej.net/people/hoerldavid">David Hörl</a>
</td></tr>

<tr valign="top">
<td><b>BigVolumeViewer Demo</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BigVolumeViewer/">https://sites.imagej.net/BigVolumeViewer/</a>
</td>
<td>Demo of BDV volume rendering. See <a href="https://forum.image.sc/t/bigvolumeviewer-tech-demo">this forum post</a> for getting started.
</td>
<td><a href="https://imagej.net/people/tpietzsch">Tobias Pietzsch</a>
</td></tr>

<tr valign="top">
<td><b>Bio-Formats</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Bio-Formats/">https://sites.imagej.net/Bio-Formats/</a>
</td>
<td>A version of <a href="https://imagej.net/formats/bio-formats">Bio-Formats</a> with the latest bugfixes from the <a href="https://github.com/openmicroscopy/bioformats/tree/develop">development branch</a>. The last SNAPSHOTS are automatically uploaded daily.
</td>
<td><a href="https://www.openmicroscopy.org">OME development team</a>
</td></tr>

<tr valign="top">
<td><b>Biomat</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Biomat/">https://sites.imagej.net/Biomat/</a>
</td>
<td>Plugins for 3D image preprocessing and detection of fibres in 3D image.
</td>
<td><a href="https://imagej.net/people/jiri-janacek">Jiri Janacek</a>
</td></tr>

<tr valign="top">
<td><b>Biomedgroup</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Biomedgroup/">https://sites.imagej.net/Biomedgroup/</a>
</td>
<td>A set of plugins developed by the biomedical imaging group of the dortmund university of applied sciences and arts: * <a href="https://imagej.net/plugins/shape-filter">Shape Filter</a> (Filter binary images by shape), * <a href="https://imagej.net/plugins/shape-smoothing">Shape Smoothing</a> (Smooth objects via Fourier descriptors), * <a href="https://imagej.net/plugins/non-local-means-denoise">Non Local Means Denoise</a> (edge preserving noise filter), * <a href="https://imagej.net/plugins/nanotrackj">NanoTrackJ</a> (Size characterization of freely diffusing nanoparticles by particle tracking), * <a href="https://imagej.net/plugins/ridge-detection">Ridge Detection</a> (Line / Ridge detection plugin).
</td>
<td><a href="https://imagej.net/people/thorstenwagner">Thorsten Wagner</a>
</td></tr>

<tr valign="top">
<td><b>BioVoxxel</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BioVoxxel/">https://sites.imagej.net/BioVoxxel/</a>
</td>
<td>A toolset providing a plugins and macros for flat-field background corrections, image filters, color coding of shape descriptors and several tools to process and analyze binary images such as the 'Speckle Inspector', 'Binary Feature Extractor' and an 'Extended Particle Analyzer' and a qualitative and quantitative evaluation tool for Fijis' Auto Threshold methods. More information at the <a href="https://imagej.net/plugins/biovoxxel-toolbox">BioVoxxel Toolbox</a> page and the <a href="https://www.biovoxxel.de">BioVoxxel web site</a>.
</td>
<td><a href="https://imagej.net/people/biovoxxel">Jan Brocher</a>
</td></tr>

<tr valign="top">
<td><b>BioVoxxel 3D Box</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/bv3dbox/">https://sites.imagej.net/bv3dbox/</a>
</td>
<td>Image processing and analysis tools for 2D and 3D images in one place. Based on GPU accelerated computing via CLIJ2. <a href="https://biovoxxel.github.io/bv3dbox/">BioVoxxel 3D box</a> <a href="https://www.biovoxxel.de">BioVoxxel web site</a>.
</td>
<td><a href="https://imagej.net/people/biovoxxel">Jan Brocher</a>
</td></tr>

<tr valign="top">
<td><b>BioVoxxel Figure Tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/bvft/">https://sites.imagej.net/bvft/</a>
</td>
<td>The figure tools are meant to connect Fiji with Inkscape to flexibly prepare images for and create publication figures with the highest possible data quality. An introduction video to version 2.2.0 can be found <a href="https://youtu.be/F6ll37NOgXc?si=SOICjiqGzw4RCUDR">here</a>
</td>
<td><a href="https://imagej.net/people/biovoxxel">Jan Brocher</a>
</td></tr>

<tr valign="top">
<td><b>Blind Analysis Tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Astja/">https://sites.imagej.net/Astja/</a>
</td>
<td>Tools to facilitate blind i.e., unbiased analysis of (image) data. For more information visit the <a href="https://imagej.net/plugins/blind-analysis-tools">documentation page</a>.
</td>
<td><a href="https://imagej.net/people/ahtsaJ">Astha Jaiswal</a>, Holger Lorenz
</td></tr>

<tr valign="top">
<td><b>BoneJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BoneJ/">https://sites.imagej.net/BoneJ/</a>
</td>
<td>A collection tools for trabecular geometry and whole bone shape analysis. This update site hosts the new experimental, modernized version of BoneJ that works with the latest Fiji. Enabling the update site creates the menu entries Plugins &gt; BoneJ. The pQCT tools are now hosted on their own update site (see below).
</td>
<td><a href="https://imagej.net/people/mdoube">Michael Doube</a>, <a href="https://imagej.net/people/alessandrofelder">Alessandro Felder</a>
</td></tr>

<tr valign="top">
<td><b>BTrack</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/BTrack/">https://sites.imagej.net/BTrack/</a>
</td>
<td><a href="https://imagej.net/plugins/btrack">BTrack</a> uses the skeletonization implementation of <a href="https://imagej.net/libs/imglib2">ImgLib2</a> to track growing ends of tissue branches. It is suitable for any form of imaging modality as it requires a segmentation image as an input for performing the tracking operations.
</td>
<td><a href="https://imagej.net/people/kapoorlab">Varun Kapoor</a>
</td></tr>

<tr valign="top">
<td><b>CALM</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CALM/">https://sites.imagej.net/CALM/</a>
</td>
<td>Plugins from the Crick Advanced Light Microscopy Facility at <a href="https://www.crick.ac.uk">The Francis Crick Institute</a>.
</td>
<td><a href="mailto:david.barry@crick.ac.uk">David Barry</a>
</td></tr>

<tr valign="top">
<td><b>CAMDU</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CAMDU/">https://sites.imagej.net/CAMDU/</a>
</td>
<td>Plugins from the <a href="https://www.warwick.ac.uk/camdu">Computing and Advanced Microscopy Development Unit</a> at the <a href="https://www.warwick.ac.uk">University of Warwick</a>.
</td>
<td><a href="mailto:e.martins-ratamero.1@warwick.ac.uk">Erick Martins Ratamero</a>
</td></tr>

<tr valign="top">
<td><b>CATS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CATS/">https://sites.imagej.net/CATS/</a>
</td>
<td>Context Aware Trainable Segmentation <a href="https://github.com/embl-cba/cats">CATS</a>
</td>
<td><a href="mailto:tischitischer@gmail.com">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>CellTrackingChallenge</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Ulman/">https://sites.imagej.net/Ulman/</a>
</td>
<td>Implementation of measures and front-end Fiji plugins used in and related to the <a href="http://www.celltrackingchallenge.net">Cell Tracking Challenge</a>. It also provides an extension for importing, editing and exporting  the results of the CellTrackingChallenge, which can be used in  conjunction with the Mastodon plugin.
</td>
<td><a href="mailto:ulman@mpi-cbg.de">Vladimír Ulman</a>
</td></tr>

<tr valign="top">
<td><b>Cellular Imaging</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Cellular-Imaging/">https://sites.imagej.net/Cellular-Imaging/</a>
</td>
<td>Fiji plugins developed in the Cellular Imaging - Lehmann Group of the FMP-Berlin.
</td>
<td><a href="mailto:christopher.schmied@fht.org">Christopher Schmied</a>
</td></tr>

<tr valign="top">
<td><b>CiliaQ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CiliaQ/">https://sites.imagej.net/CiliaQ/</a>
</td>
<td>An set of three ImageJ plugins to quantify ciliary shape, length, and fluorescence in 2D, 3D, and 4D images. For more details, a link to the corresponding publication, and information on how to cite the plugin visit the CiliaQ github pages (including manual and wiki) at https://github.com/hansenjn/CiliaQ
</td>
<td><a href="https://github.com/hansenjn/">Jan Hansen</a>
</td></tr>

<tr valign="top">
<td><b>CIP</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CIP/">https://sites.imagej.net/CIP/</a>
</td>
<td><a href="https://imagej.net/plugins/cip">An image processing toolbox focused on scripting</a>.
</td>
<td><a href="https://imagej.net/people/benoalo">Benoit Lombardot</a>
</td></tr>

<tr valign="top">
<td><b>CircleSkinner</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CircleSkinner/">https://sites.imagej.net/CircleSkinner/</a>
</td>
<td>A Fiji plugin for the automated detection and quantification of circular structures in images.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>ClearVolume</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ClearVolume/">https://sites.imagej.net/ClearVolume/</a>
</td>
<td>ClearVolume is a fast 3D viewer, for more information see also the <a href="https://github.com/ClearVolume/ClearVolume/wiki">ClearVolume wiki</a>.
</td>
<td><a href="https://imagej.net/User:ClearVolume">ClearVolume</a>
</td></tr>

<tr valign="top">
<td><b>clij</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/clij/">https://sites.imagej.net/clij/</a>
</td>
<td>GPU-acceleration for ImageJ. See <a href="https://clij.github.io/">https://clij.github.io/</a>
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>clij2</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/clij2/">https://sites.imagej.net/clij2/</a>
</td>
<td>GPU-acceleration for ImageJ. Please also activate "clij" if you want to use "clij2". See <a href="https://clij.github.io/">https://clij.github.io/</a>
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>clijx-assistant</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/clijx-assistant/">https://sites.imagej.net/clijx-assistant/</a>
</td>
<td>Experimental update site for the future release of clij2. See <a href="https://clij.github.io/assistant">https://clij.github.io/assistant</a>
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>clijx-assistant-extensions</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/clijx-assistant-extensions/">https://sites.imagej.net/clijx-assistant-extensions/</a>
</td>
<td>Experimental update site for the future release of clij2. See <a href="https://clij.github.io/assistant">https://clij.github.io/assistant</a>
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>CMCI-EMBL</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Miura/">https://sites.imagej.net/Miura/</a>
</td>
<td>A plugin for loading sample images and misc functionality for CMCI ImageJ course. See <a href="https://wiki.cmci.info/mainpages/courses">CMCI course</a>. Eventually more plugins will be added.
</td>
<td><a href="https://imagej.net/people/miura">Kota Miura</a>
</td></tr>

<tr valign="top">
<td><b>CMP-BIA tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CMP-BIA/">https://sites.imagej.net/CMP-BIA/</a>
</td>
<td>The development version of <a href="https://imagej.net/plugins/cmp-bia-tools">CMP-BIA tools</a>.
</td>
<td><a href="http://cmp.felk.cvut.cz/~borovji3/">Jiri Borovec</a>
</td></tr>

<tr valign="top">
<td><b>CMTK Registration</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Jefferis/">https://sites.imagej.net/Jefferis/</a>
</td>
<td>CMTK Registration GUI.
</td>
<td><a href="https://imagej.net/people/jefferis">Gregory Jefferis</a>
</td></tr>

<tr valign="top">
<td><b>Colocalization by Cross Correlation</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Amccall/">https://sites.imagej.net/Amccall/</a>
</td>
<td>A co-localization plugin that attempts to find non-random spatial correlations between two images and provide an estimate of their distance and standard deviation. Conceptually similar to Van Steensel's CCF. For more details see our <a href="https://imagej.net/plugins/colocalization-by-cross-correlation">wiki page</a>.
</td>
<td><a href="https://imagej.net/people/andmccall">Andrew McCall</a>
</td></tr>

<tr valign="top">
<td><b>ColonyArea</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ColonyArea/">https://sites.imagej.net/ColonyArea/</a>
</td>
<td>A plugin that is optimized to perform standard analysis of colony formation assays conducted in 6- to 24-well dishes. The plugin processes each well individually and determines not the colony number, but the area of the well covered with cells, also taking the intensity into account.
</td>
<td><a href="https://bioimaging.fi">Turku BioImaging</a>, <a href="https://github.com/junelsolis">Junel Solis</a>
</td></tr>

<tr valign="top">
<td><b>ColorBlindLUTs</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ColorBlindLUTs/">https://sites.imagej.net/ColorBlindLUTs/</a>
</td>
<td><a href="https://github.com/bruvellu/color-blind-luts">ColorBlindLUTs</a> is a set of color blind friendly lookup tables for Fiji/ImageJ.
</td>
<td><a href="https://brunovellutini.com/">Bruno C. Vellutini</a>
</td></tr>

<tr valign="top">
<td><b>Colour Deconvolution2</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ColourDeconvolution2/">https://sites.imagej.net/ColourDeconvolution2/</a>
</td>
<td>Update site for <a href="https://blog.bham.ac.uk/intellimic/g-landini-software/colour-deconvolution-2/">Colour Deconvolution2</a>
</td>
<td><a href="https://imagej.net/people/landinig">Gabriel Landini</a>
</td></tr>

<tr valign="top">
<td><b>Cookbook</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Cookbook/">https://sites.imagej.net/Cookbook/</a>
</td>
<td>Plugins from the <a href="https://imagej.net/imaging">Cookbook</a>.
</td>
<td><a href="https://imagej.net/imaging#Credits">Cookbook Team</a>
</td></tr>

<tr valign="top">
<td><b>CSBDeep</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CSBDeep/">https://sites.imagej.net/CSBDeep/</a>
</td>
<td>Plugins for executing pretrained CARE networks.
</td>
<td><a href="https://imagej.net/people/frauzufall">Deborah Schmidt</a>
</td></tr>

<tr valign="top">
<td><b>CSIM Laboratory</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Acsenrafilho/">https://sites.imagej.net/Acsenrafilho/</a>
</td>
<td>A set of plugins, macros and scripts developed by the CSIM (Computing in Signal and Images in Medicine) research group. One of the methods available are <a href="https://imagej.net/plugins/anomalous-diffusion-filters">Anomalous Diffusion Filters</a>, <a href="https://imagej.net/plugins/noise-generator">Noise Generator</a>, <a href="https://imagej.net/plugins/mri-perfusion">MRI Perfusion</a> and the <a href="https://imagej.net/plugins/sampen2d">Sample Entropy for Images</a>. Please, see the <a href="https://imagej.net/plugins/csim-lab">CSIM group web page</a> to find out how the methods works and more information about us.
</td>
<td><a href="https://imagej.net/people/acsenrafilho">Antonio Carlos da S. Senra Filho</a>
</td></tr>

<tr valign="top">
<td><b>CWNS dense nuclei segmenter</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/CWNS/">https://sites.imagej.net/CWNS/</a>
</td>
<td>The Fiji implementation of the CWNS dense nuclei segmenter, presented in <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0150853">"Object Segmentation and Ground Truth in 3D Embryonic Imaging"</a>, by B. Rajasekaran et al., Plos ONE, 2016.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>DeepClas4Bio-plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Adines/">https://sites.imagej.net/Adines/</a>
</td>
<td>Collection of plugins that use <a href="https://github.com/adines/DeepClas4Bio">DeepClas4Bio API</a> -- an API that facilitates the interoperability of bioimaging tools with deep learning frameworks.
</td>
<td><a href="mailto:adines@unirioja.es">Adrián Inés</a>
</td></tr>

<tr valign="top">
<td><b>DeepImageJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/DeepImageJ/">https://sites.imagej.net/DeepImageJ/</a>
</td>
<td>Bioimage processing using (pre-)trained deep learning models from TensorFlow or PyTorch. The collection of deepImageJ plugins enables the interaction with the BioImage Model Zoo. Further details are available in the <a href="https://deepimagej.github.io/deepimagej/">deepImageJ web page</a>.
</td>
<td><a href="https://github.com/esgomezm">Estibaliz Gómez de Mariscal</a>, <a href="mailto:daniel.sage@epfl.ch">Daniel Sage</a>, <a href="mailto:mamunozb@ing.uc3m.es">Arrate Muñoz-Barrutia</a>
</td></tr>

<tr valign="top">
<td><b>Dendrite Dynamics Tracker</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/DendriteDynamicsTracker/">https://sites.imagej.net/DendriteDynamicsTracker/</a>
</td>
<td>Dendrite-Dynamics-Tracker is a plugin based on <a href="https://imagej.net/plugins/trackmate/">TrackMate</a> and <a href="https://imagej.net/plugins/analyze-skeleton/index">AnalyzeSkeleton</a> that can measure the dynamics of dendrites by tracking their tip over time.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>DHM Utilities</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Sudgy/">https://sites.imagej.net/Sudgy/</a>
</td>
<td>Collection of plugins for use in digital holographic microscopy, such as hologram reconstruction, numerical propagation, and phase unwrapping.
</td>
<td><a href="mailto:dcohoe@pdx.edu">David Cohoe</a>
</td></tr>

<tr valign="top">
<td><b>DiameterJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/DiameterJ/">https://sites.imagej.net/DiameterJ/</a>
</td>
<td>Macros for quantitative analysis of nanofiber images. DiameterJ is able to analyze any network of connecting lines (validated for SEM micrographs of micro- and nanofibers) and produce metrics such as mean diameter, histograms of fiber diameter, mean pore size, histogram of all pore areas, percent porosity, etc. See <a href="coming">soon</a> for more information and the publication describing how DiameterJ works and its limitations.
</td>
<td><a href="https://imagej.net/people/NHotaling">Nathan Hotaling</a>
</td></tr>

<tr valign="top">
<td><b>DiversePathsJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Vuhlmann/">https://sites.imagej.net/Vuhlmann/</a>
</td>
<td>DiversePathsJ implements a dynamic programming approach for the general task of finding diverse shortest paths between two end-points. It is not linked to a specific biological problem and can be applied to a large variety of images. See the <a href="http://bigwww.epfl.ch/algorithms/diversepathsj/">DiversePathsJ web page</a> for more information.
</td>
<td><a href="https://imagej.net/people/vuhlmann">Virginie Uhlmann</a>
</td></tr>

<tr valign="top">
<td><b>DNA FISH</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/DNA-FISH/">https://sites.imagej.net/DNA-FISH/</a>
</td>
<td>Plugin for semi-automated <a href="https://github.com/embl-cba/DNA-FISH#analyse-fish-spots---fiji-plugin">analysis of DNA FISH spots</a>.
</td>
<td>Christian Tischer
</td></tr>

<tr valign="top">
<td><b>Draw 3D ROI</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Draw3DROI/">https://sites.imagej.net/Draw3DROI/</a>
</td>
<td>A Fiji plugin to create 3D masks from hand-drawn 2D ROIs. 
</td>
<td><a href="https://imagej.net/people/andmccall">Andrew McCall</a>
</td></tr>

<tr valign="top">
<td><b>ElastixWrapper</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ElastixWrapper/">https://sites.imagej.net/ElastixWrapper/</a>
</td>
<td>Image registration using <a href="http://elastix.isi.uu.nl/">elastix</a>
</td>
<td><a href="mailto:christian.tischer@embl.de">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>ELEPHANT</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ELEPHANT/">https://sites.imagej.net/ELEPHANT/</a>
</td>
<td><a href="https://elephant-track.github.io">ELEPHANT</a> is a platform for 3D cell tracking, based on incremental and interactive deep learning. ELEPHANT is implemented as a plugin for <a href="https://github.com/mastodon-sc/mastodon">Mastodon</a>. Please not that users need to set up the ELEPHANT server in addition to this plugin. The detailed instructions can be found <a href="https://elephant-track.github.io">here</a>.
</td>
<td><a href="https://github.com/ksugar">Ko Sugawara</a>
</td></tr>

<tr valign="top">
<td><b>EM Tool</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IMBalENce/">https://sites.imagej.net/IMBalENce/</a>
</td>
<td><a href="https://imagej.net/plugins/imbalence">EM tool</a> is a suite of simple tools for electron microscopist
</td>
<td><a href="mailto:zhou.xu@monash.edu">Zhou Xu</a>
</td></tr>

<tr valign="top">
<td><b>EpiGraph</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Pedgomgal1/">https://sites.imagej.net/Pedgomgal1/</a>
</td>
<td>EpiGraph: an open-source method to quantify epithelial organization. EpiGraph is a Fiji plugin that combines computational geometry and graph theory to measure organization in any computational or natural tessellation image.
</td>
<td><a href="mailto:pgomez-ibis@us.es">Pedro J. Gómez Gálvez</a>, <a href="https://github.com/Pablo1990">Pablo Vicente Munuera</a>
</td></tr>

<tr valign="top">
<td><b>EVAnalyzer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/evanalyzer/">https://sites.imagej.net/evanalyzer/</a>
</td>
<td><a href="https://github.com/joda01/evanalyzer">EVAnalyzer</a> EVAnalyzer is a Fiji ImageJ plugin designed for analyzing microscope pictures of EVs. This plugin can be used for: Counting the number of EVs; Calculating the colocalization of evs and other particles; Detect biological cells and counting EVs updatekn from cells; Analysis of a large amount of images automatically;Generating XLSX (Excel) reports with automatic statistics generation.
</td>
<td><a href="mailto:joachim.danmayr@mailbox.org">Joachim Danmayr</a>
</td></tr>

<tr valign="top">
<td><b>Excel Functions</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ExcelMacroExtensions/">https://sites.imagej.net/ExcelMacroExtensions/</a>
</td>
<td>The Excel Functions and Excel Macro Extensions are meant to build an interface beween ImageJ and Excel (.xlsx). It serves as a library which can be used via the ImageJ macro language, Java and Jython to facilitate data transfer <a href="https://gitlab.com/biovoxxel/excel-table-macro-extensions">Excel Macro Extensions</a>.
</td>
<td><a href="https://imagej.net/people/biovoxxel">Jan Brocher</a>
</td></tr>

<tr valign="top">
<td><b>EZFig</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/EZF/">https://sites.imagej.net/EZF/</a>
</td>
<td>EZFig is a tool to build publication-ready scientific figures based on ScientiFig but simpler to use. It must be used in combination with the 'Java-8' update site.
</td>
<td><a href="mailto:scientifig@gmail.com">Benoit Aigouy</a>
</td></tr>

<tr valign="top">
<td><b>FAST</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FAST/">https://sites.imagej.net/FAST/</a>
</td>
<td>Fluorescence image AnalysiS Tool automates tasks to process a set of single-channel fluorescence images.
</td>
<td><a href="https://imagej.net/people/cyrilturies">C.TURIÈS</a>
</td></tr>

<tr valign="top">
<td><b>Fast4DReg</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Fast4DReg/">https://sites.imagej.net/Fast4DReg/</a>
</td>
<td><a href="https://imagej.net/plugins/fast4dreg">Fast4DReg</a> is a Fiji macro for drift correction of 3D videos or channel alignment in 3D multichannel image stacks. Drift or misalignment can be corrected in all x-, y- and/or z-directions.
</td>
<td><a href="https://imagej.net/people/jpylvanainen">Joanna Pylvänäinen</a>
</td></tr>

<tr valign="top">
<td><b>FFMPEG</b>
</td>
<td><a rel="nofollow" class="external free" href="https://fiji.sc/~schindelin/ffmpeg-plugins/">https://fiji.sc/~schindelin/ffmpeg-plugins/</a>
</td>
<td>Fiji plugins which allow reading and writing to <a href="https://www.ffmpeg.org/">FFmpeg</a>-supported formats via native bindings.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>FibrilJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FibrilJ/">https://sites.imagej.net/FibrilJ/</a>
</td>
<td>Macros developed to measure the diameters and persistence lengths if amyloids fibrils by AFM, EM, STM etc. iamges. It provides histogram distribution of the average fibril diameter and average fibril persistence length. Was validated for AFM, SEM and TEM images of Sup35NM fibrils on mica, silicon, gold surfaces and formvar film.
</td>
<td>Petr Sokolov
</td></tr>

<tr valign="top">
<td><b>Filopodyan</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Rb7777/">https://sites.imagej.net/Rb7777/</a>
</td>
<td>Filopodia dynamics analysis. See <a href="https://github.com/gurdon-institute/Filopodyan">GitHub</a> for the user guide, source code and R scripts for downstream analysis.
</td>
<td>Richard Butler, Vasja Urbančič
</td></tr>

<tr valign="top">
<td><b>FiloQuant</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FiloQuant/">https://sites.imagej.net/FiloQuant/</a>
</td>
<td><a href="https://imagej.net/people/guijacquemet">FiloQuant</a> is a user friendly and modifiable tool for automated detection and quantification of filopodia properties such as length and density. FiloQuant can also track filopodia.
</td>
<td>Guillaume Jacquemet, Alexandre Carisey
</td></tr>

<tr valign="top">
<td><b>FiubaMate</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FiubaMate/">https://sites.imagej.net/FiubaMate/</a>
</td>
<td><a href="https://github.com/franco-giordano/fiuba-taller3-final">FiubaMate</a> is a detection and tracking system for bodies in channels, built for 'Medios Porosos' lab, FIUBA.
</td>
<td><a href="mailto:jferres@fi.uba.ar">Julian Ferres</a>, <a href="mailto:fgiordano@fi.uba.ar">Franco Giordano</a>
</td></tr>

<tr valign="top">
<td><b>FLIMJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FLIMJ/">https://sites.imagej.net/FLIMJ/</a>
</td>
<td><a href="https://imagej.net/plugins/flimj">FLIMJ</a> is a plugin for analyzing FLIM data in ImageJ, using the <a href="https://flimlib.github.io/">FLIMLib</a> library.
</td>
<td>Dasong Gao, Curtis Rueden
</td></tr>

<tr valign="top">
<td><b>FlyReg</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FlyReg/">https://sites.imagej.net/FlyReg/</a>
</td>
<td><a href="https://github.com/tischi/fly-reg">FlyReg</a> is a plugin for the 3D registration of fly (drosophila) embryos.
</td>
<td>Christian Tischer
</td></tr>

<tr valign="top">
<td><b>FOCUST</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FOCUST/">https://sites.imagej.net/FOCUST/</a>
</td>
<td><a href="https://sebastianamos.github.io/FOCUST-Plugin-Site/">FOCUST</a> is the Fluorescent Object and Cell gpU-accelerated Segmentation Toolbox.
</td>
<td><a href="https://github.com/SebastianAmos">Sebastian Amos</a>
</td></tr>

<tr valign="top">
<td><b>FPBioimage</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Fpbioimage/">https://sites.imagej.net/Fpbioimage/</a>
</td>
<td>A plugin to prepare and upload data for <a href="https://imagej.net/plugins/fpbioimage">Fpbioimage</a>.
</td>
<td><a href="https://imagej.net/User:Fpbioimage">FPBioimage</a>
</td></tr>

<tr valign="top">
<td><b>FracLac Suite</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Akarperien/">https://sites.imagej.net/Akarperien/</a>
</td>
<td>Fractal Analysis and Modelling tools for neuroscience and other disciplines. <strong>Analysis:</strong> <abbr title=\"Box Counting Fractal Dimension\">D<sub>B</sub></abbr>, <abbr title=\"Fixed Grid and Sliding Box\">lacunarity</abbr>, multifractal spectra, <abbr title=\"Local Connected Fractal Dimension\">D<sub>LC</sub></abbr>, <abbr title=\"Mass vs Distance Dimension\">D<sub>MvD</sub></abbr>, subscanning, and graphical wave analysis for binary and gray scale images, with inline and <a href="https://imagej.net/plugins/fraclac/FLHelp/Introduction.htm">online</a> theoretical and practical help. <strong>Outputs:</strong> detailed analyses, raw data, plots, colour-coding, and settings (*.fl) files. <strong>Modelling:</strong> benchmark fractals, natural structures (e.g., neurons), megadata to graphical wave images. Code in jar. See <a href="https://imagej.net/plugins/fraclac/FLHelp/FLCitations.htm">citations</a>.
</td>
<td><a href="https://imagej.net/User:Akarperien">A. Karperien</a>
</td></tr>

<tr valign="top">
<td><b>FreQ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FreQ/">https://sites.imagej.net/FreQ/</a>
</td>
<td>An ImageJ plugin to analyze pixel intensity oscillations in time-lapse 2D images. For example, this allows to characterize the ciliary beat of multi-ciliated cells in time-lapse images acquired with a bright-field or phase-contrast microscope. For more details, a link to the corresponding publication, and information on how to cite the plugin visit the FreQ github pages at https://github.com/hansenjn/FreQ
</td>
<td><a href="https://github.com/hansenjn/">Jan Hansen</a>
</td></tr>

<tr valign="top">
<td><b>FRET Imaging</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FRET-Imaging/">https://sites.imagej.net/FRET-Imaging/</a>
</td>
<td>ImageJ plugins for 3-cube FRET (<a href="https://imagej.net/plugins/rifret">RiFRET</a>) and acceptor photobleaching FRET (<a href="https://imagej.net/plugins/accpbfret">AccPbFRET</a>) from <a href="https://github.com/CellMoTher">CellMoTher</a>.
</td>
<td><a href="https://imagej.net/people/camlloyd">Cameron Lloyd</a>
</td></tr>

<tr valign="top">
<td><b>FunImageJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/FunImageJ/">https://sites.imagej.net/FunImageJ/</a>
</td>
<td>Fiji plugin that provides Clojure wrappings of many commonly used functions from ImageJ and ImgLib2. Works with SciJava scripting for Clojure. FunImageJ scripts can be converted into standalone ImageJ/Fiji-based programs. Code available <a href="https://github.com/kephale/fun.imagej">on GitHub</a>.
</td>
<td><a href="https://imagej.net/people/kephale">Kyle Harrington</a>
</td></tr>

<tr valign="top">
<td><b>Fuzzy logic and artificial neural networks image processing toolbox for ImageJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Astartes91/">https://sites.imagej.net/Astartes91/</a>
</td>
<td>Toolbox includes fuzzy edge detection plugin, fuzzy contrast enhancement plugin and ann edge detection plugin. Source code <a href="https://github.com/astartes91/imagej-neural-fuzzy-plugins">on GitHub</a>.
</td>
<td><a href="mailto:astartes91@gmail.com">Vladimir Nizamutdinov</a>
</td></tr>

<tr valign="top">
<td><b>Fuzzy Set</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Rerger/">https://sites.imagej.net/Rerger/</a>
</td>
<td><a href="https://imagej.net/plugins/fuzzy-set">Fuzzy Set</a> is an ImageJ plugin, to transform the intensity of the image using Fuzzy Sets.
</td>
<td><a href="mailto:rerger@leomail.tamuc.edu">Robert Erger</a>
</td></tr>

<tr valign="top">
<td><b>GDSC</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/GDSC/">https://sites.imagej.net/GDSC/</a>
</td>
<td>The Genome Damage and Stability Centre (GDSC) plugins are a collection of analysis programs for microscopy images including colocalisation analysis and peak finding (FindFoci). See <a href="http://www.sussex.ac.uk/gdsc/intranet/microscopy/UserSupport/AnalysisProtocol/imagej/gdsc_plugins/">GDSC Plugins</a> for details.
</td>
<td><a href="mailto:a.herbert@sussex.ac.uk">Alex Herbert</a>
</td></tr>

<tr valign="top">
<td><b>GDSC-SMLM2</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/GDSC-SMLM2/">https://sites.imagej.net/GDSC-SMLM2/</a>
</td>
<td>Tools for Single Molecule Light Microscopy (SMLM) including PALM, STORM and other single molecule microscopy methods. See the latest documentation on <a href="https://gdsc-smlm.readthedocs.io/">ReadTheDocs</a>.
</td>
<td><a href="mailto:a.herbert@sussex.ac.uk">Alex Herbert</a>
</td></tr>

<tr valign="top">
<td><b>Gut Analysis Toolbox</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/GutAnalysisToolbox/">https://sites.imagej.net/GutAnalysisToolbox/</a>
</td>
<td><a href="https://imagej.net/plugins/gut-analysis-toolbox">Gut Analysis Toolbox</a> enables semi-automated analysis of the neuronal and glial distribution within the gut wall in 2D. It uses a combination of Fiji macros, StarDist and deepImageJ for automated segmentation. The primary focus is on characterising the enteric nervous system. The aim is to expand the analysis to other celltypes within the gut and enable 3D analysis.
</td>
<td><a href="https://imagej.net/people/pr4deepr">Pradeep Rajasekhar</a>
</td></tr>

<tr valign="top">
<td><b>Hadim</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Hadim/">https://sites.imagej.net/Hadim/</a>
</td>
<td>Contains the ij2-python-lib package.
</td>
<td><a href="https://imagej.net/people/hadim">Hadrien Mary</a>
</td></tr>

<tr valign="top">
<td><b>HDF5</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Ronneber/">https://sites.imagej.net/Ronneber/</a>
</td>
<td>The HDF5 plugin allows to load and save data in HDF5 format. For details see <a href="https://lmb.informatik.uni-freiburg.de/resources/opensource/imagej_plugins/hdf5.html">HDF5 plugin</a>.
</td>
<td><a href="mailto:ronneber@informatik.uni-freiburg.de">Olaf Ronneberger</a>
</td></tr>

<tr valign="top">
<td><b>HistoJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Pathomation/">https://sites.imagej.net/Pathomation/</a>
</td>
<td>Plugin to interface with <a href="https://www.pathomation.com/">Pathomation's digital slide viewer</a> which can open Olympus VSI (.vsi), Ventana / Roche BIF (.bif), Hamamatsu NDPI (.ndpi), 3DHistech MRXS (.mrxs), Aperio / Leica SVS (.svs), Leica SCN (.scn), Carl Zeiss ZVI (.zvi), Carl Zeiss CZI (.czi), Nikon ND2 (.nd2), Sakura SVSlide (.svslide).
</td>
<td><a href="mailto:info@pathomation.com">Pathomation</a>
</td></tr>

<tr valign="top">
<td><b>Hohlbein Lab</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/HohlbeinLab/">https://sites.imagej.net/HohlbeinLab/</a>
</td>
<td>Plugins created by the <a href="https://jhohlbein.com/">Hohlbein Lab</a> (Wageningen University, Laboratory of Biophysics). Includes ThunderSTORM with added <a href="https://aip.scitation.org/doi/abs/10.1063/1.5005899">phasor-based localization</a> for single-molecule microscopy analysis (pSMLM-3D, Martens et al., JCP, 2018).
</td>
<td><a href="mailto:koen.martens@wur.nl">Koen Martens</a>, <a href="mailto:johannes.hohlbein@wur.nl">Johannes Hohlbein</a>
</td></tr>

<tr valign="top">
<td><b>HPC-Datastore</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/HPC-Datastore/">https://sites.imagej.net/HPC-Datastore/</a>
</td>
<td>Plugins using HPC Datastore.
</td>
<td><a href="mailto:jan@kozusznik.cz">Jan Kožusznik</a>
</td></tr>

<tr valign="top">
<td><b>HPC-ParallelTools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/HPC-ParallelTools/">https://sites.imagej.net/HPC-ParallelTools/</a>
</td>
<td>Plugins using parallelization and HPC.
</td>
<td><a href="mailto:jan@kozusznik.cz">Jan Kožusznik</a>
</td></tr>

<tr valign="top">
<td><b>IamMM</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IamMM/">https://sites.imagej.net/IamMM/</a>
</td>
<td><a href="https://imagej.net/plugins/image-synthesizer">Image Synthesizer</a> (plugin to generate images from mathematical functions and conditional statements in a coordinate system independent from the aspect ratio or image size), <a href="https://github.com/IamMM/Spheroid_RGB">Spheroid RGB</a> (plugin to analyze microscopy RGB images with fluorescent stained cells)
</td>
<td><a href="mailto:m.maske@posteo.de Maximilian Maske">Maximilian Maske</a>
</td></tr>

<tr valign="top">
<td><b>IBMP-CNRS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Mutterer/">https://sites.imagej.net/Mutterer/</a>
</td>
<td><a href="https://imagejdocu.tudor.lu/doku.php?id=plugin:utilities:figurej:start">FigureJ</a> (easy article figure creation), <a href="https://imagejdocu.tudor.lu/doku.php?id=plugin:utilities:action_bar:start">ActionBar</a> (macro-based buttons bars), <a href="https://imagejdocu.tudor.lu/doku.php?id=plugin:utilities:droplet:start">Droplet</a> (drag and drop file processor), LSM Reader and LSM Toolbox moved to the main Fiji update site, <a href="https://imagejdocu.tudor.lu/doku.php?id=plugin:utilities:ij_ed:start">IJ_Ed</a> (jEdit link).
</td>
<td><a href="mailto:jerome.mutterer@gmail.com">Jerome Mutterer</a>
</td></tr>

<tr valign="top">
<td><b>IJ-OpenCV-plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IJ-OpenCV/">https://sites.imagej.net/IJ-OpenCV/</a>
</td>
<td>Collection of plugins created using the <a href="https://github.com/joheras/IJ-OpenCV">IJ-OpenCV library</a> -- a library that allows the connection of ImageJ and OpenCV.
</td>
<td><a href="mailto:joheras@gmail.com">Jónathan Heras</a>
</td></tr>

<tr valign="top">
<td><b>IJ-Plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IJ-Plugins/">https://sites.imagej.net/IJ-Plugins/</a>
</td>
<td>Plugins from <a href="https://github.com/ij-plugins/ij-plugins-bundle">IJ-Plugins</a> <a href="https://github.com/ij-plugins/ij-plugins-bundle">bundle</a> project, include <a href="https://github.com/ij-plugins/ijp-color">color calibration</a> and conversions, <a href="https://github.com/ij-plugins/ijp-toolkit/wiki/Segmentation">segmentation algorithms</a>, <a href="https://github.com/ij-plugins/ijp-toolkit/wiki/Filters">various anisotropic diffucion filters</a>, <a href="https://github.com/ij-plugins/ijp-DeBayer2SX">raw image processing</a>, additional <a href="https://github.com/ij-plugins/ijp-imageio">2D</a> and <a href="https://github.com/ij-plugins/ijp-toolkit/wiki/3D-IO">3D</a> file formats support, and <a href="https://github.com/ij-plugins/ij-plugins-bundle">more</a>.
</td>
<td><a href="mailto:jpsacha@gmail.com">Jarek Sacha</a>
</td></tr>

<tr valign="top">
<td><b>IJMMD</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IJMMD/">https://sites.imagej.net/IJMMD/</a>
</td>
<td><a href="https://github.com/haesleinhuepf/imagejmacromarkdown">ImageJ Macro Markdown</a> scripting language interpreter to create notebook-like pages using ImageJ Macro and Markdown.
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>IJPB-plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IJPB-plugins/">https://sites.imagej.net/IJPB-plugins/</a>
</td>
<td>Collection of plugins created at the INRA-IJPB Modeling and Digital Imaging lab to perform 2D/3D image segmentation, fast morphological operations and geometric analysis
</td>
<td><a href="https://www-ijpb.versailles.inra.fr/en/bc/equipes/modelisation-imagerie/">Modeling and Digital Imaging lab, INRA-Versailles</a>
</td></tr>

<tr valign="top">
<td><b>ilastik</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Ilastik/">https://sites.imagej.net/Ilastik/</a>
</td>
<td>Provides interoperability between Fiji and ilastik workflows (pixel/object classification, autocontext, multicut, tracking) as well as HDF5 file export/import.
</td>
<td><a href="mailto:team@ilastik.org">ilastik team</a>
</td></tr>

<tr valign="top">
<td><b>ImageJ Latex</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Yul.liuyu/">https://sites.imagej.net/Yul.liuyu/</a>
</td>
<td>Add math equations to an image using Latex.
</td>
<td><a href="https://imagej.net/people/ohliumliu">Yu Liu</a>
</td></tr>

<tr valign="top">
<td><b>ImageJ-ITK</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ImageJ-ITK/">https://sites.imagej.net/ImageJ-ITK/</a>
</td>
<td>Adds <a href="https://github.com/imagej/imagej-itk">ITK integration</a> including conversion between <a href="http://www.simpleitk.org/">ITK</a> and <a href="https://github.com/imagej/imagej">ImageJ</a> images. See the <a href="https://imagej.net/software/itk">ITK page</a> for more information.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>ImageJ-MATLAB</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MATLAB/">https://sites.imagej.net/MATLAB/</a>
</td>
<td>Adds support for MATLAB <a href="https://github.com/scijava/scripting-matlab">scripting</a> and converting between MATLAB arrays and <a href="https://github.com/imagej/imagej-matlab">ImageJ</a> components. See the <a href="https://imagej.net/scripting/matlab">scripting tutorial</a> for more information.
</td>
<td><a href="https://imagej.net/people/hinerm">Mark Hiner</a>
</td></tr>

<tr valign="top">
<td><b>ImageScience</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ImageScience/">https://sites.imagej.net/ImageScience/</a>
</td>
<td>Base update site for all <a href="https://imagej.net/libs/imagescience">ImageScience</a> related ImageJ plugins.
</td>
<td><a href="https://imagej.net/people/emeijering">Erik Meijering</a>
</td></tr>

<tr valign="top">
<td><b>ImagingFCS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ImagingFCS/">https://sites.imagej.net/ImagingFCS/</a>
</td>
<td>Imaging<em>FCS 1.491 is a basic ImageJ plugin to calculate and view spatio-temporal correlation functions from 16 bit gray tiff stack files. It was written under Fiji (ImageJ 1.51f; Java 1.8.0</em>102) and requires Imagescience for statistics (simulator) and Apache Poi for file reading and writing.
</td>
<td><a href="http://www.dbs.nus.edu.sg/lab/BFL/imfcs_image_j_plugin.html">Thorsten Wohland</a>
</td></tr>

<tr valign="top">
<td><b>IMCF Uni Basel</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/UniBas-IMCF/">https://sites.imagej.net/UniBas-IMCF/</a>
</td>
<td>Various plugins (most notably the Stitching-Wrapper for Olympus FluoView OIF / OIB mosaics), macros and Action Bars, including a LUT selection bar that lets you click on a visual representation of the LUT. Also ships the <a href="https://github.com/miura/MultiStackRegistration">MultiStackRegistration</a> plugin (the version modified by Kota Miura that is usable from scripts).
</td>
<td><a href="https://imagej.net/people/ehrenfeu">Niko Ehrenfeuchter</a>
</td></tr>

<tr valign="top">
<td><b>ImgLib2-Imaris-Bridge</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ImgLib2-Imaris-Bridge/">https://sites.imagej.net/ImgLib2-Imaris-Bridge/</a>
</td>
<td><a href="https://imaris.github.io/imglib2-imaris-bridge">ImgLib2-Imaris-Bridge</a> provides sharing arbitrarily large images between Imaris and Fiji/ImageJ2 without duplicating memory requirements. Imaris datasets are represented as cached images in ImgLib2, lazily loaded via Imaris XT, and exposed through an ImageJ2 service.
</td>
<td><a href="https://imagej.net/people/tpietzsch">Tobias Pietzsch</a>
</td></tr>

<tr valign="top">
<td><b>IMOD-ZOLA</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IMOD-ZOLA/">https://sites.imagej.net/IMOD-ZOLA/</a>
</td>
<td>Zernike Optimized Localization Algorithm for 3D single molecule localizations <a href="https://github.com/imodpasteur/ZOLA-3D">ZOLA-3D</a> developed in Institut Pasteur, group IMOD -- a CUDA-enabled localization algorithm for extended PSF single-molecule localization microscopy. Paper is under review.
</td>
<td><a href="mailto:aaristov@pasteur.fr">Andrey Aristov</a>, <a href="mailto:benoit.lelandais@pasteur.fr">Benoit Lelandais</a>
</td></tr>

<tr valign="top">
<td><b>INCF test site</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Dimiterpp/">https://sites.imagej.net/Dimiterpp/</a>
</td>
<td>Update site for scale space filters and other INCF projects from GSOC 2014, 2015, 2016.
</td>
<td><a href="mailto:dimiterpp@gmail.com">Dimiter Prodanov</a>
</td></tr>

<tr valign="top">
<td><b>Intensity Profile tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IntensityProfileTools/">https://sites.imagej.net/IntensityProfileTools/</a>
</td>
<td>Provides a macro-toolset for interactive inspection of images using intensity profile plots. Including an interactive mouse tool (line profiles) and an action tool for the generation of horizontal/vertical average intensity panels.
</td>
<td><a href="https://imagej.net/people/LauLauThom">Laurent Thomas</a>
</td></tr>

<tr valign="top">
<td><b>IsletJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/IsletJ/">https://sites.imagej.net/IsletJ/</a>
</td>
<td>Plugin for evaluation of microscopy images of Langerhans islets. Provides estimate of area and count of islets, and an estimate of purity (endocrine-to-exocrine tissue ratio).
</td>
<td><a href="mailto:schier@utia.cas.cz">Jan Schier</a>, <a href="mailto:Jan.Svihlik@vscht.cz">Jan Švihlík</a>
</td></tr>

<tr valign="top">
<td><b>JAX Eye</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MarkKrebs/">https://sites.imagej.net/MarkKrebs/</a>
</td>
<td>Macro for processing brightfield fundus images of the mouse eye and plugin for averaging replicate OCT volume scans.
</td>
<td><a href="mailto:mark.krebs@jax.org">Mark Krebs</a>
</td></tr>

<tr valign="top">
<td><b>JDM LUTs</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/JDM_LUTs/">https://sites.imagej.net/JDM_LUTs/</a>
</td>
<td>A collection of lookup tables, including some designed for high dynamic range data.
</td>
<td><a href="mailto:jmanton@mrc-lmb.cam.ac.uk">James Manton</a>
</td></tr>

<tr valign="top">
<td><b>JIPipe</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/JIPipe/">https://sites.imagej.net/JIPipe/</a>
</td>
<td><a href="https://imagej.net/plugins/jipipe">JIPipe</a> is a visual programming language based on ImageJ.
</td>
<td><a href="mailto:ruman.gerst@leibniz-hki.de">Ruman Gerst</a>, <a href="mailto:zoltan.cseresnyes@leibniz-hki.de">Zoltán Cseresnyés</a>, <a href="mailto:thilo.figge@leibniz-hki.de">Marc Thilo Figge</a>
</td></tr>

<tr valign="top">
<td><b>JOGL Canvas Deep Color / 3D</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/JOGLCanvas/">https://sites.imagej.net/JOGLCanvas/</a>
</td>
<td><a href="https://github.com/aschain/JOGL_Canvas_DC3D">JOGL Canvas</a> is a plugin for ImageJ1 that uses OpenGL and a graphics card to display images rather than the ImageCanvas class, which is extended from java's canvas. It replaces it with a <a href="https://jogamp.org/jogl/www/">JOGL</a> OpenGL GLCanvas instead. This allows 10-bit or greater color depth for supported monitors and some modest 3D rendering.
</td>
<td><a href="https://imagej.net/User:JOGLCanvas">Aaron Schain</a>
</td></tr>

<tr valign="top">
<td><b>Kappa</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Kappa/">https://sites.imagej.net/Kappa/</a>
</td>
<td>A Fiji plugin for Curvature Analysis.
</td>
<td><a href="https://imagej.net/people/hadim">Hadrien Mary</a>
</td></tr>

<tr valign="top">
<td><b>KTZ-LUTs</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/KTZ-LUTs/">https://sites.imagej.net/KTZ-LUTs/</a>
</td>
<td>Handmade collection of <a href="https://github.com/kwolbachia/KTZ-LUTs">LUTs</a> for microscopy, scientific visualization and art.
</td>
<td><a href="mailto:kevinterretaz@gmail.com">Kevin Terretaz</a>
</td></tr>

<tr valign="top">
<td><b>Lab-utility-plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Georgenm/">https://sites.imagej.net/Georgenm/</a>
</td>
<td>A collection of plugins for file blinding, batch image conversion of proprietary formats to tif, and other common helper functions. Source and docs <a href="https://github.com/Macklin-Lab/imagej-microscopy-scripts">on GitHub</a>.
</td>
<td><a href="mailto:nicholas.m.george@ucdenver.edu">Nick George</a>
</td></tr>

<tr valign="top">
<td><b>Labkit</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Labkit/">https://sites.imagej.net/Labkit/</a>
</td>
<td>An advanced plugin for labeling and automatic segmentaion of Images.
</td>
<td><a href="mailto:arzt@mpi-cbg.de">Matthias Arzt</a>
</td></tr>

<tr valign="top">
<td><b>LAG SIM</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/LagSIM/">https://sites.imagej.net/LagSIM/</a>
</td>
<td>A plugin and set of macro tools to help with reconstructing SIM data. This has been specifically developed for the Laser Analytics Group SIM in the University of Cambridge - but please download and/or modify for your own needs.
</td>
<td><a href="https://imagej.net/User:LagSIM">LagSIM</a>
</td></tr>

<tr valign="top">
<td><b>Leiden University</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Willemsejj/">https://sites.imagej.net/Willemsejj/</a>
</td>
<td>At <a href="https://www.universiteitleiden.nl/">Leiden University</a> we develop user demanded plugins for general use, which could be applicable in your experiments as well. Most questions arise from experiments done in the <a href="https://www.universiteitleiden.nl/en/research/research-facilities/science/cell-observatory#tab-1">Cell Observatory</a>.
</td>
<td><a href="mailto:jwillemse@biology.leidenuniv.nl">Joost Willemse</a>
</td></tr>

<tr valign="top">
<td><b>LimeSeg</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/LimeSeg/">https://sites.imagej.net/LimeSeg/</a>
</td>
<td>Revisited active contour method for 3D segmentation. Documentation: <a href="https://imagej.net/plugins/limeseg">LimeSeg</a>.
</td>
<td><a href="mailto:nicolas.chiaruttini@epfl.ch">Nicolas Chiaruttini</a>, Sarah Machado
</td></tr>

<tr valign="top">
<td><b>Lineage-Mapper</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NIST-ISG-LineageMapper/">https://sites.imagej.net/NIST-ISG-LineageMapper/</a>
</td>
<td>Lineage Mapper is an open-source, overlap-based cell tracking system for time-lapse images of biological cells, colonies, and particles. Lineage Mapper tracks objects independently of the segmentation method, detects mitosis in confluence, separates cell clumps mistakenly segmented as a single cell, provides accuracy and scalability even on terabyte-sized datasets, and creates division and/or fusion lineages.
</td>
<td><a href="https://isg.nist.gov/deepzoomweb/resources/csmet/pages/cell_tracking/cell_tracking.html">NIST ISG</a>
</td></tr>

<tr valign="top">
<td><b>Live-Kymographer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Live-Kymographer/">https://sites.imagej.net/Live-Kymographer/</a>
</td>
<td>Generate kymographs with a live preview.
</td>
<td><a href="mailto:remi.berthoz+imagej@gmail.com">Rémi Berthoz</a>
</td></tr>

<tr valign="top">
<td><b>liveim-tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/liveim-tools/">https://sites.imagej.net/liveim-tools/</a>
</td>
<td><a href="https://sites.imagej.net/liveim-tools/">liveim-tools</a> Macros and plugins of the MPIBPC live-cell imaging facility to process data generated by facility microscopes (e.g. Abberior, Zeiss, Visitron).
</td>
<td><a href="mailto:apoliti@mpibpc.mpg.de">Antonio Politi</a>
</td></tr>

<tr valign="top">
<td><b>LLTT</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Alex-krull/">https://sites.imagej.net/Alex-krull/</a>
</td>
<td>The <a href="https://imagej.net/plugins/low-light-tracking-tool">Low Light Tracking Tool</a> can be used to track low-light sub-resolution objects in microscopy images and other applications.
</td>
<td><a href="https://imagej.net/User:Alex">Alexander Krull</a>, <a href="https://imagej.net/people/damienrj">Damien Ramunno-Johnson</a>
</td></tr>

<tr valign="top">
<td><b>LMCF-IMG</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/LMCF-IMG/">https://sites.imagej.net/LMCF-IMG/</a>
</td>
<td>Plugins developed for work at <a href="https://www.img.cas.cz/core-facilities/light-microscopy/">LMCF IMG</a>, including <a href="https://imagej.net/plugins/wavelet-denoise">Wavelet Denoise</a>.
</td>
<td><a href="https://imagej.net/people/LMCF-IMG">Martin Capek</a>
</td></tr>

<tr valign="top">
<td><b>Local Z Projector</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/LocalZProjector/">https://sites.imagej.net/LocalZProjector/</a>
</td>
<td><a href="https://gitlab.pasteur.fr/iah-public/localzprojector">Local Z Projector</a> is an ImageJ2 plugin to perform local-Z projection of a 3D stack, possibly over time, possibly very large.
</td>
<td><a href="https://research.pasteur.fr/en/member/jean-yves-tinevez/">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>LOCI</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/LOCI/">https://sites.imagej.net/LOCI/</a>
</td>
<td>The <a href="https://loci.wisc.edu/software/fiji-plugins">LOCI</a> update site for <a href="https://loci.wisc.edu/software/fiji-plugins">their Fiji plugins</a>, including <a href="https://loci.wisc.edu/software/visbio-fiji-plugins">VisBio Fiji plugins</a> and other miscellany.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>LSM-W2</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Bionet.nsc/">https://sites.imagej.net/Bionet.nsc/</a>
</td>
<td>Plugin extracts Leaf Surface Morphology from Laser Scanning Microscopy images. The plugin can work with multi-channel multi-frame 3D images in *.lsm format obtained from a laser scanning microscope. Has a wide functionality for working with a similar type of image. For more information <a href="https://imagej.net/plugins/lsm-worker">see documentation</a>. Source code <a href="https://github.com/JmanJ/LSM_Worker">here</a>.
</td>
<td>Verman Pavel
</td></tr>

<tr valign="top">
<td><b>LUMoS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Tmcrae/">https://sites.imagej.net/Tmcrae/</a>
</td>
<td>Plugin performs blind spectral unmixing on multichannel images, particularly for fluorescence microscopy. For more information <a href="https://imagej.net/plugins/lumos-spectral-unmixing">see documentation</a>. Source code <a href="https://github.com/tristan-mcrae-rochester/Multiphoton-Image-Analysis/tree/master/Spectral%20Unmixing/Code/ImageJ-FIJI">here</a>.
</td>
<td>Tristan McRae
</td></tr>

<tr valign="top">
<td><b>LysoQuant</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/LysoQuant/">https://sites.imagej.net/LysoQuant/</a>
</td>
<td>LysoQuant is a deep learning approach for the automatic segmentation and classification of fluorescence images capturing cargo delivery within endolysosomes. For more information <a href="https://www.irb.usi.ch/lysoquant/">see documentation</a>. Source code <a href="https://github.com/irb-imagingfacility/lysoquant">here</a>.
</td>
<td><a href="mailto:diego.morone@irb.usi.ch">Diego Morone</a>
</td></tr>

<tr valign="top">
<td><b>MaMuT</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MaMuT/">https://sites.imagej.net/MaMuT/</a>
</td>
<td><a href="https://imagej.net/plugins/mamut">MaMuT</a>, a plugin for the annotation of massive, multi-view data.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>Mars</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Mars/">https://sites.imagej.net/Mars/</a>
</td>
<td><a href="https://duderstadt-lab.github.io/mars-docs/">Mars</a>, a platform for storage, exploration, and processing of image-derived properties of biomolecules together with enhanced OME metadata.
</td>
<td><a href="mailto:duderstadt@biochem.mpg.de">Karl Duderstadt</a>
</td></tr>

<tr valign="top">
<td><b>Maskflow</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Maskflow/">https://sites.imagej.net/Maskflow/</a>
</td>
<td>A Fiji plugin for Mask RCNN semantic segmentation.
</td>
<td><a href="https://imagej.net/people/hadim">Hadrien Mary</a>
</td></tr>

<tr valign="top">
<td><b>Masks from ROIs</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MasksfromRois/">https://sites.imagej.net/MasksfromRois/</a>
</td>
<td>Add the command Mask(s) from Roi(s) to Edit &gt; Selection. Generate mask images (0/255) from Rois in the RoiManager. Compatible with image-stacks too. Ideal for the generation of ground-truth segmentation masks.
</td>
<td><a href="https://imagej.net/people/LauLauThom">Laurent Thomas</a>
</td></tr>

<tr valign="top">
<td><b>Mastodon</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Mastodon-jungle/">https://sites.imagej.net/Mastodon-jungle/</a>
</td>
<td><a href="https://github.com/mastodon-sc/mastodon">Mastodon</a> is a software for automatic, semi-automatic and manual cell tracking in very large images. It is meant to be an interactive tool able to harness a large amount of annotations. You can see it as the successor of MaMuT and TrackMate on steroids.
</td>
<td><a href="https://imagej.net/people/tpietzsch">Tobias Pietzsch</a>, <a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>Mastodon-DeepLineage</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Mastodon-DeepLineage/">https://sites.imagej.net/Mastodon-DeepLineage/</a>
</td>
<td><a href="https://github.com/mastodon-sc/mastodon-deep-lineage">Mastodon-DeepLineage</a> provides an extension for the Mastodon plugin and can only be used in conjunction with it. Mastodon allows tracking cell lineages in 3D+t images. Mastodon-DeepLineage extends this with a collection of tools to analyse lineages of tracked objects.
</td>
<td><a href="https://imagej.net/people/stefanhahmann">Stefan Hahmann</a>, <a href="https://imagej.net/people/maarzt">Matthias Arzt</a>
</td></tr>

<tr valign="top">
<td><b>Mastodon-Tomancak</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Mastodon-Tomancak/">https://sites.imagej.net/Mastodon-Tomancak/</a>
</td>
<td><a href="https://github.com/mastodon-sc/mastodon">Mastodon-Tomancak</a> provides extensions  for the Mastodon plugin and can only be used in conjunction with it. Mastodon allows tracking cell lineages in 3D+t images. Mastodon-Tomancak extends this with a collection of tools to edit, analyse and visualise lineages of tracked objects.
</td>
<td><a href="https://imagej.net/people/maarzt">Matthias Arzt</a>, <a href="https://imagej.net/people/stefanhahmann">Stefan Hahmann</a>, <a href="https://imagej.net/people/xulman">Vladimír Ulman</a>
</td></tr>

<tr valign="top">
<td><b>Mcat</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Mcat/">https://sites.imagej.net/Mcat/</a>
</td>
<td>The MSOT cluster analysis toolkit <a href="https://github.com/applied-systems-biology/mcat">Mcat</a> is a software for objective and quantitative analysis of multispectral optoacoustic tomography (MSOT) image data. It is based on a signal-oriented analysis approach, i.e. the entire sample is analysed regarding the signal kinetics of the molecule of interest and pixel-wise kinetic clustering is performed to extract quantitative information about the pharmacokinetics of the investigated molecule and its biodistribution in the sample.
</td>
<td><a href="https://www.leibniz-hki.de/en/applied-systems-biology.html">Research Group Applied Systems Biology, HKI, Jena</a>
</td></tr>

<tr valign="top">
<td><b>MiC mask comparator</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MiC-mask-comparator/">https://sites.imagej.net/MiC-mask-comparator/</a>
</td>
<td><a href="https://github.com/Multimodal-Imaging-Center/MiC">MiC</a> is a plugin to compare segmentation mask to a reference one. the comparison is available at pixel or object's level. It computes the number of true positives, false positives and false negatives and provides metrics for quantitative estimation of segmentation quality. It displays several result tables containing all information, plots and superposition of masks for visual inspection. The plugins works on stacks (only 2D comparison) and is macro recordable.
</td>
<td><a href="https://institut-curie.org/platform/curiecoretech-multimodal-imaging-center-uar2016-us43">Cedric Messaoudi</a>
</td></tr>

<tr valign="top">
<td><b>MIC-MAQ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MIC-MAQ/">https://sites.imagej.net/MIC-MAQ/</a>
</td>
<td>[MIC-MAQ] (https://github.com/MultimodalImagingCenter/MIC-MAQ) is a plugin for automatic segmentation of nuclei and/or cells for quantifications in other channels including foci detection. MIC-MAQ automatically segments cells and/or nuclei on 2D microscopy images using thresholding or deep-learning approaches. It takes Z-stack but works only on 2D projection. It provides intensity based and/or morphological measurements in nuclei and/or cells in multi channels experiment. The plugin propose the foci/spots detection inside nuclei/cells. the plugins works on all images present in working directory (any image format that can be read by bioformat).
</td>
<td><a href="https://institut-curie.org/platform/curiecoretech-multimodal-imaging-center-uar2016-us43">Cedric Messaoudi</a>
</td></tr>

<tr valign="top">
<td><b>Micro-Magellan</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Hpinkard/">https://sites.imagej.net/Hpinkard/</a>
</td>
<td><a href="https://micro-manager.org/wiki/MicroMagellan">Micro-Magellan</a> is a plugin based on Micro-Manager for high throughput microscopy. This update site is used to enable reading of Micro-Magellan datasets by the <a href="https://imagej.net/plugins/bdv">BigDataViewer</a>.
</td>
<td>Henry Pinkard
</td></tr>

<tr valign="top">
<td><b>Microglia-Morphometry</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Microglia-Morphometry/">https://sites.imagej.net/Microglia-Morphometry/</a>
</td>
<td><a href="https://github.com/embl-cba/microglia-morphometry#microglia-morphometry">Microglia-Morphometry</a> is a tool for the semi-automated segmentation, tracking and morphometric analysis of 2D microglia time-lapse images.
</td>
<td><a href="mailto:christian.tischer@embl.de">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>Mighty Data, Inc.</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/mightydatainc/">https://sites.imagej.net/mightydatainc/</a>
</td>
<td><a href="https://github.com/Mighty-Data-Inc/dendritic-spine-counter">Dendritic Spine Counter</a> creates a modular wizard dialog to assist with the task of counting dendritic spines in microscope images using open-source statistical image processing techniques.
</td>
<td>Mikhail Voloshin
</td></tr>

<tr valign="top">
<td><b>MIST</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NIST-ISG-MIST/">https://sites.imagej.net/NIST-ISG-MIST/</a>
</td>
<td>Microscopy Image Stitching Tool (MIST), is a stitching algorithm for small and large two dimensional image grid collections. We developed a new and novel method for optimizing the translations computed by the phase-correlation method using the Fourier transform approach. This new method estimates the microscope stage repeatability from the computed translations of a gridded image tiles. It then optimizes all translation computation using the Hill Climbing algorithm bounded to four times the stage repeatability. This minimizes the maximum uncertainty related to the translation computation for any pair of images.
</td>
<td><a href="https://isg.nist.gov/deepzoomweb/resources/csmet/pages/image_stitching/image_stitching.html">NIST ISG</a>
</td></tr>

<tr valign="top">
<td><b>MiToBo</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MiToBo/">https://sites.imagej.net/MiToBo/</a>
</td>
<td><a href="https://mitobo.informatik.uni-halle.de/">MiToBo</a> is a collection of basic image processing operators (filtering, morphology, labeling, etc.), advanced analysis algorithms (wavelets, snakes, levelsets, etc.), and special applications mainly dedicated to microscopy image analysis (nuclei detection, scratch assay analysis, neurite image segmentation, etc.).
</td>
<td><a href="https://mitobo.informatik.uni-halle.de/">The MiToBo Team, Halle, Germany</a>
</td></tr>

<tr valign="top">
<td><b>MoBIE</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MoBIE/">https://sites.imagej.net/MoBIE/</a>
</td>
<td>Multimodal Big Image Data Exploration and Sharing <a href="https://github.com/mobie-org/mobie">MoBIE</a>
</td>
<td><a href="mailto:tischitischer@gmail.com">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>ModularImageAnalysis (MIA)</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ModularImageAnalysis/">https://sites.imagej.net/ModularImageAnalysis/</a>
</td>
<td>MIA is a plugin which provides a modular framework for assembling image and object analysis workflows. Detected objects can be transformed, filtered, measured and related. Analysis workflows are batch-enabled by default, allowing easy processing of high-content datasets.  For more information please see the documentation at <a href="https://mianalysis.github.io/">mianalysis.github.io</a> or on <a href="https://imagej.net/plugins/modularimageanalysis/">imagej.net</a>.
</td>
<td><a href="https://imagej.net/people/sjcross">Stephen Cross</a>
</td></tr>

<tr valign="top">
<td><b>Molography</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/VolkerGatterdam/">https://sites.imagej.net/VolkerGatterdam/</a>
</td>
<td>Script for Evaluation of Molographic Data
</td>
<td><a href="https://imagej.net/User:VolkerGatterdam">Volker Gatterdam</a>
</td></tr>

<tr valign="top">
<td><b>MoMA</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MoMA/">https://sites.imagej.net/MoMA/</a>
</td>
<td><a href="https://github.com/fjug/MoMA/wiki">Mother Machine Analyser</a>
</td>
<td><a href="https://imagej.net/people/fjug">Florian Jug</a>
</td></tr>

<tr valign="top">
<td><b>Morphology</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Landini/">https://sites.imagej.net/Landini/</a>
</td>
<td>The update site for the <a href="https://blog.bham.ac.uk/intellimic/g-landini-software/">Mathematical Morphology Collection</a> of plugins and macros.
</td>
<td><a href="https://imagej.net/people/landinig">Gabriel Landini</a>
</td></tr>

<tr valign="top">
<td><b>MOSAIC ToolSuite</b>
</td>
<td><a rel="nofollow" class="external free" href="https://mosaic.mpi-cbg.de/Downloads/update/Fiji/MosaicToolsuite">https://mosaic.mpi-cbg.de/Downloads/update/Fiji/MosaicToolsuite</a>
</td>
<td>An integrated toolbox containing the following image-processing software from the MOSAIC Group: 2D and 3D single-particle tracking, 2D and 3D multi-region image segmentation, 2D and 3D globally optimal two-region image segmentation, point-spread function estimation in 2D and 3D, background subtraction, Poisson noise simulator, microscope optics simulator, chromatic aberration correction.
</td>
<td><a href="https://mosaic.mpi-cbg.de">MOSAIC Group, MPI-CBG, Dresden</a>
</td></tr>

<tr valign="top">
<td><b>MotiQ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MotiQ/">https://sites.imagej.net/MotiQ/</a>
</td>
<td>An set of ImageJ plugins to quantify dynamics, morphology, and fluorescence of microglial (or other) cells in 2D and 3D (also time-lapse) images. For more details, a link to the corresponding publication, and information on how to cite the plugins visit the MotiQ github pages at https://github.com/hansenjn/MotiQ
</td>
<td><a href="https://github.com/hansenjn/">Jan Hansen</a>
</td></tr>

<tr valign="top">
<td><b>MPIBPC</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MPIBPC/">https://sites.imagej.net/MPIBPC/</a>
</td>
<td>MPIBPC stands for Max Planck Institute for Biophysical Chemistry. We provide the Imspector OBF reader for the Bio-Formats plugin.
</td>
<td><a href="https://www.mpibpc.mpg.de/en">MPIBPC</a>, <a href="https://www.mpibpc.mpg.de/hell">NanoBiophotonics</a>, <a href="https://www.mpibpc.mpg.de/person/11861/102807">Bjoern Thiel</a>
</td></tr>

<tr valign="top">
<td><b>MS-ECS-2D</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MS-ECS-2D/">https://sites.imagej.net/MS-ECS-2D/</a>
</td>
<td>MembraneStained EpithelialCellSegmentation 2D, for more information see also the <a href="https://imagej.net/plugins/ms-ecs-2d-update-site">update site page</a>.
</td>
<td><a href="https://imagej.net/plugins/ms-ecs-2d-update-site">MS-ECS-2D</a>
</td></tr>

<tr valign="top">
<td><b>MTrack</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MicrotubuleTracker/">https://sites.imagej.net/MicrotubuleTracker/</a>
</td>
<td>MTrack is a tool, which detects, tracks, and measures the behavior of fluorescently labeled microtubules imaged by TIRF (total internal reflection fluorescence) microscopy. In such an in vitro reconstitution approach, stabilized, non-dynamic microtubule seeds serve as nucleation points for dynamically growing microtubules.
</td>
<td><a href="https://imagej.net/people/kapoorlab">Varun Kapoor</a>
</td></tr>

<tr valign="top">
<td><b>Multi-Template-Matching</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Multi-Template-Matching/">https://sites.imagej.net/Multi-Template-Matching/</a>
</td>
<td><a href="https://imagej.net/plugins/multi-template-matching">Multiple template matching</a> allows to easily localize one or multiple objects in images using one or a set of template images. The template images can be flipped and/or rotated to increase the chance to find the objects.
</td>
<td><a href="https://imagej.net/people/LauLauThom">Laurent Thomas</a>
</td></tr>

<tr valign="top">
<td><b>Multifrac</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Multifrac/">https://sites.imagej.net/Multifrac/</a>
</td>
<td><a href="https://sites.imagej.net/Multifrac">Multifrac</a> is a tool for multifractal and scaling analysis of 2D and 3D stack images, including lacunarity, configuration entropy, characteristic lenght and multifractal spectrum of gray and black and white images. This is particularly desginated and tested for addressing analysis on soil pore structre.
</td>
<td><a href="https://imagej.net/User:Ivangtorre">Ivan G Torre</a>
</td></tr>

<tr valign="top">
<td><b>MultiStackReg</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MultiStackReg/">https://sites.imagej.net/MultiStackReg/</a>
</td>
<td><a href="https://imagej.net/plugins/multistackreg">MultiStackReg</a> does registration of a stack based on the registration results of another stack. It uses Turboreg as the backend.
</td>
<td><a href="https://imagej.net/people/miura">Kota Miura</a>
</td></tr>

<tr valign="top">
<td><b>N5</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/N5/">https://sites.imagej.net/N5/</a>
</td>
<td><a href="https://github.com/saalfeldlab/n5">N5</a> libraries.
</td>
<td><a href="mailto:saalfelds@janelia.hhmi.org">Stephan Saalfeld</a>
</td></tr>

<tr valign="top">
<td><b>NanoJ-Core</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NanoJ/">https://sites.imagej.net/NanoJ/</a>
</td>
<td>The update site providing NanoJ's core functionality. <a href="https://bitbucket.org/rhenriqueslab/nanoj-core">NanoJ</a> is a set of high-performance libraries for GPU enabled super-resolution data analysis.
</td>
<td><a href="https://www.ucl.ac.uk/lmcb/users/ricardo-henriques">The Henriques Team, UCL, UK</a>
</td></tr>

<tr valign="top">
<td><b>NanoJ-SQUIRREL</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NanoJ-SQUIRREL/">https://sites.imagej.net/NanoJ-SQUIRREL/</a>
</td>
<td><a href="https://bitbucket.org/rhenriqueslab/nanoj-squirrel">NanoJ-SQUIRREL</a> is an analytical toolbox to measure super-resolution image quality and quantitatively map defects, it also provides methods to aid in improving super-resolution images.
</td>
<td><a href="https://www.ucl.ac.uk/lmcb/users/ricardo-henriques">The Henriques Team, UCL, UK</a>
</td></tr>

<tr valign="top">
<td><b>NanoJ-SRRF</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NanoJ-SRRF/">https://sites.imagej.net/NanoJ-SRRF/</a>
</td>
<td><a href="https://bitbucket.org/rhenriqueslab/nanoj-srrf">NanoJ-SRRF</a> is a high-performance analytical approach for Live-cell Super-Resolution Microscopy, provided as a fast GPU-enabled ImageJ plugin.
</td>
<td><a href="https://www.ucl.ac.uk/lmcb/users/ricardo-henriques">The Henriques Team, UCL, UK</a>
</td></tr>

<tr valign="top">
<td><b>NanoJ-VirusMapper</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NanoJ-VirusMapper/">https://sites.imagej.net/NanoJ-VirusMapper/</a>
</td>
<td><a href="https://bitbucket.org/rhenriqueslab/nanoj-virusmapper">NanoJ-VirusMapper</a> is a high-throughput ImageJ plugin for single-particle analysis in fluorescence microscopy, particularly super-resolution microscopy.
</td>
<td><a href="https://www.ucl.ac.uk/lmcb/users/ricardo-henriques">The Henriques Team, UCL, UK</a>
</td></tr>

<tr valign="top">
<td><b>NEUBIAS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Neubias/">https://sites.imagej.net/Neubias/</a>
</td>
<td>Plugins and scripts developed in the context of <a href="http://neubias.org">NEUBIAS</a>.
</td>
<td><a href="http://neubias.org">NEUBIAS members</a>
</td></tr>

<tr valign="top">
<td><b>Neuroanatomy</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Neuroanatomy/">https://sites.imagej.net/Neuroanatomy/</a>
</td>
<td>Installs <a href="https://imagej.net/plugins/snt/">SNT</a>, the ImageJ framework for quantification of neuronal anatomy
</td>
<td><a href="https://imagej.net/people/tferr">Tiago Ferreira</a>
</td></tr>

<tr valign="top">
<td><b>NeuroCyto Lab</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Christopheleterrier/">https://sites.imagej.net/Christopheleterrier/</a>
</td>
<td>Various tools from the NeuroCyto Lab in Marseille for image processing and quantification, including SMLM scripts. Centered on the analysis of fluorescence in neuronal cultures.
</td>
<td>Christophe Leterrier, <a href="http://www.neurocytolab.org">NeuroCyto Lab</a>, INP, Marseille
</td></tr>

<tr valign="top">
<td><b>NeuroCyto LUTs</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NeuroCyto-LUTs/">https://sites.imagej.net/NeuroCyto-LUTs/</a>
</td>
<td>A collection of LUTs packaged as an easy to use extra menu. LUTs are from various sources detailled in the About section.
</td>
<td>Christophe Leterrier, <a href="http://www.neurocytolab.org">NeuroCyto Lab</a>, INP, Marseille
</td></tr>

<tr valign="top">
<td><b>NoiSee</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/NoiSee/">https://sites.imagej.net/NoiSee/</a>
</td>
<td>An easy-to-use macro for measuring and analyzing SNR (signal-to-noise ratio) of light microscopes. See the <a href="https://imagej.net/plugins/noisee">NoiSee</a> page for more details.
</td>
<td><a href="https://imagej.net/people/ehrenfeu">Niko Ehrenfeuchter</a>
</td></tr>

<tr valign="top">
<td><b>Nscale-segmentation</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Nscale_segmentation/">https://sites.imagej.net/Nscale_segmentation/</a>
</td>
<td>A customizable segmentation tool for fluorescence cell microscopy analysis used to separate clustered/touching nuclei. See the <a href="https://imagej.net/plugins/nscale-segmentation">Nscale-segmentation</a> page for more information.
</td>
<td><a href="edentamrat@gmail.com">Eden Tamrat</a>
</td></tr>

<tr valign="top">
<td><b>Nuclear-Medicine</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Nuclear-Medicine/">https://sites.imagej.net/Nuclear-Medicine/</a>
</td>
<td>Conventional nuclear medicine applications and additional packages for the <a href="http://petctviewer.org">PET/CT Viewer</a>.
</td>
<td>Salim Kanoun, Toulouse, France
</td></tr>

<tr valign="top">
<td><b>NucleusJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PouletAxel/">https://sites.imagej.net/PouletAxel/</a>
</td>
<td>A plugin for quantifying nuclear shape and chromatin organization. Segments nuclear boundaries and nuclear bodies, then automatically measures parameters quantifying shape and size of nuclei as well as intra-nuclear objects and the positioning of the objects within the nuclear volume.
</td>
<td>Axel Poulet, Genetic Reproduction and Development lab, Clermont-Ferrand, France
</td></tr>

<tr valign="top">
<td><b>OMERO 5.0</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/OMERO-5.0/">https://sites.imagej.net/OMERO-5.0/</a>
</td>
<td>The <a href="https://openmicroscopy.org/info/omero">OMERO</a> update site for <a href="https://github.com/imagej/imagej-omero">bidirectional integration between ImageJ and OMERO 5.0.x</a>.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>OMERO 5.1</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/OMERO-5.1/">https://sites.imagej.net/OMERO-5.1/</a>
</td>
<td>The <a href="https://openmicroscopy.org/info/omero">OMERO</a> update site for <a href="https://github.com/imagej/imagej-omero">bidirectional integration between ImageJ and OMERO 5.1.x</a>.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>OMERO 5.2</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/OMERO-5.2/">https://sites.imagej.net/OMERO-5.2/</a>
</td>
<td>The <a href="https://openmicroscopy.org/info/omero">OMERO</a> update site for <a href="https://github.com/imagej/imagej-omero">bidirectional integration between ImageJ and OMERO 5.2.x</a>.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>OMERO 5.4</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/OMERO-5.4/">https://sites.imagej.net/OMERO-5.4/</a>
</td>
<td>The <a href="https://openmicroscopy.org/info/omero">OMERO</a> update site for <a href="https://github.com/imagej/imagej-omero">bidirectional integration between ImageJ and OMERO 5.4.x</a>.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>OMERO 5.5-5.6</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/OMERO-5.5-5.6/">https://sites.imagej.net/OMERO-5.5-5.6/</a>
</td>
<td>The <a href="https://openmicroscopy.org/info/omero">OMERO</a> update site for <a href="https://github.com/imagej/imagej-omero">bidirectional integration between ImageJ and OMERO 5.5.x and 5.6.x</a>.
</td>
<td><a href="https://imagej.net/people/hinerm">Mark Hiner</a>, <a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>OpenMIMS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/OpenMIMS2/">https://sites.imagej.net/OpenMIMS2/</a>
</td>
<td>OpenMIMS is an ImageJ plugin to open, process and analyze images captured with NanoSIMS 50 &amp; 50L secondary ion mass spectrometers (Cameca).
</td>
<td><a href="mailto:cni@partners.org">Matt Steinhauser</a>, <a href="mailto:wrtaylor@partners.org">Walter Taylor</a>
</td></tr>

<tr valign="top">
<td><b>OpenSPIM</b>
</td>
<td><a rel="nofollow" class="external free" href="https://openspim.org/update/">https://openspim.org/update/</a>
</td>
<td>The drivers and software to operate an <a href="https://openspim.org/">OpenSPIM</a> (Open Access selective plane light sheet microscope).
</td>
<td>UNSUPPORTED
</td></tr>

<tr valign="top">
<td><b>OPTIMISME</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Dbenielli/">https://sites.imagej.net/Dbenielli/</a>
</td>
<td>Optimism is an ImageJ plugin that implements a deconvolution method based on the Majorization minimization algorithm, devoted to two-photon 3d microscopic images.
</td>
<td>Dominique Benielli, <a href="https://www.i2m.univ-amu.fr/">I2M, Marseille</a>, Caroline Chaux Moulin, <a href="https://www.i2m.univ-amu.fr/">I2M, Marseille</a>, Sandrine Anthoine, <a href="https://www.i2m.univ-amu.fr/">I2M, Marseille</a>, Labex Archimede, <a href="https://www.i2m.univ-amu.fr/">I2M, Marseille</a>
</td></tr>

<tr valign="top">
<td><b>Oxford Oncology</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/MicroscopyOncologyOxford/">https://sites.imagej.net/MicroscopyOncologyOxford/</a>
</td>
<td>Description: Some basic tools developed for users of the Microscopy Core in the department of Oncology at the University of Oxford.
</td>
<td><a href="mailto:rhodri.wilson@oncology.ox.ac.uk">Rhodri Wilson</a>
</td></tr>

<tr valign="top">
<td><b>ParticleSizer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Ndef-psizer/">https://sites.imagej.net/Ndef-psizer/</a>
</td>
<td>A <a href="https://imagej.net/plugins/particlesizer">plugin to derive number based size distributions</a> from TEM images.
</td>
<td><a href="https://imagej.net/people/thorstenwagner">Thorsten Wagner</a>
</td></tr>

<tr valign="top">
<td><b>Pendent Drop</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Daerr/">https://sites.imagej.net/Daerr/</a>
</td>
<td>A <a href="https://fiji.sc/Pendent_Drop">plugin for capillary length measurement</a> through the pendent drop method.
</td>
<td><a href="http://www.msc.univ-paris-diderot.fr/~daerr/">Adrian Daerr</a>
</td></tr>

<tr valign="top">
<td><b>PET-CT</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Ilan/">https://sites.imagej.net/Ilan/</a>
</td>
<td>Update site for the Pet-Ct-Viewer and other Nuclear Medicine software in use at Beth Israel Hospital, see <a href="https://sourceforge.net/p/bifijiplugins/wiki/Pet-Ct%20Viewer/">Help</a>
</td>
<td><a href="https://imagej.net/people/ilan">Ilan Tal</a>
</td></tr>

<tr valign="top">
<td><b>PHANTAST</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Nicjac/">https://sites.imagej.net/Nicjac/</a>
</td>
<td>Update site for <a href="https://github.com/nicjac/PHANTAST-FIJI">PHANTAST</a> (phase contrast microscopy segmentation toolbox), a collection of tools for the processing of phase contrast microscopy images
</td>
<td><a href="http://www.nicjac.net">Nicolas Jaccard</a>
</td></tr>

<tr valign="top">
<td><b>PhotoBend</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Anotherche/">https://sites.imagej.net/Anotherche/</a>
</td>
<td>The <a href="https://imagej.net/plugins/photobend">PhotoBend</a> is a collection of specialized plugins providing tracking a needle-like crystal shape changing during photoinduced bending process. 1. tracking the curvature of uniformly curved rods changing with time; 2. tracking a laser spot movement used in the laser beam deflection technique. Additionally, two auxiliary plugins for reading and writing compressed video using FFmpeg are included, which can be used standalone.
</td>
<td><a href="https://imagej.net/people/anotherche">Stanislav Chizhik</a>
</td></tr>

<tr valign="top">
<td><b>PhotonImaging</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PhotonImaging/">https://sites.imagej.net/PhotonImaging/</a>
</td>
<td>The <a href="https://imagej.net/plugins/photon-imaging">PhotonImaging</a> plug-in for ImageJ is able to process single photon event data, by locating the center point of each photon and create a combined greyscale image with all found photons per pixel mapped to the correct pixel value.
</td>
<td><a href="mailto:lonsch96@live.nl">Lonneke Scheffer</a>, <a href="mailto:wout.van.helvoirt@gmail.com">Wout van Helvoirt</a>
</td></tr>

<tr valign="top">
<td><b>PillarTracker</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PillarTracker/">https://sites.imagej.net/PillarTracker/</a>
</td>
<td>The <a href="https://imagej.net/plugins/pillartracker">PillarTracker</a> plug-in enables user to track the pillars and visualize the tracks and deflections.
</td>
<td><a href="mailto:mbixxc@nus.edu.sg">XU Xiaochun</a>, <a href="mailto:mbifmm@nus.edu.sg">Felix Martin Margadant</a>
</td></tr>

<tr valign="top">
<td><b>PIM-IBIS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Abastien/">https://sites.imagej.net/Abastien/</a>
</td>
<td>Update site for the <a href="https://microscopie.ibis.ulaval.ca/">IBIS Microscopy &amp; Imaging Platform</a> at <a href="https://www.ulaval.ca/">ULaval</a>.
</td>
<td><a href="https://imagej.net/people/alexandrebastien">Alexandre Bastien</a>
</td></tr>

<tr valign="top">
<td><b>PixFRET</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PixFRET/">https://sites.imagej.net/PixFRET/</a>
</td>
<td>PixFRET - an ImageJ plug-in for FRET calculation which can accommodate variations in spectral bleed-throughs
</td>
<td><a href="https://imagej.net/people/imagejan">Jan Eglinger</a>
</td></tr>

<tr valign="top">
<td><b>PlateViewer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PlateViewer/">https://sites.imagej.net/PlateViewer/</a>
</td>
<td>PlateViewer - an ImageJ plug-in for visualisation of high-throughput microscopy multi-well plate image data.
</td>
<td><a href="mailto:christian.tischer@embl.de">Christian Tischer</a>
</td></tr>

<tr valign="top">
<td><b>PQCT</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PQCT/">https://sites.imagej.net/PQCT/</a>
</td>
<td>Tools for Stratec <a href="https://imagej.net/pQCT">pQCT</a> bone and soft tissue images.
</td>
<td>Timo Rantalainen
</td></tr>

<tr valign="top">
<td><b>PTBIOP</b>
</td>
<td><a rel="nofollow" class="external free" href="https://biop.epfl.ch/Fiji-Update">https://biop.epfl.ch/Fiji-Update</a>
</td>
<td>The <a href="https://biop.epfl.ch/">EPFL - PTBIOP</a> (BioImaging &amp; Optics Platform) Update Site, with Fiji plugins, macros and ActionBars. We recommend that you also activate the <strong>IBMP-CNRS</strong>, <strong>ImageScience</strong> and <em>3D ImageJ Suite</em> update sites. Update site contents at <a href="https://wiki-biop.epfl.ch/en/ipa/fiji/update-site#ptbiop-update-site-contents">https://wiki-biop.epfl.ch/en/ipa/fiji/update-site#ptbiop-update-site-contents</a>
</td>
<td><a href="https://people.epfl.ch/olivier.burri?lang=en&cvlang=en">Olivier Burri</a>
</td></tr>

<tr valign="top">
<td><b>Puncta Process</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/puncta_process/">https://sites.imagej.net/puncta_process/</a>
</td>
<td>The <a href="https://imagej.net/plugins/process-puncta/">Puncta Process PlugIn</a> is a practical analysis tool for counting puncta in 3D image data. The process uses single-channel data to produce three specific values: - <strong>Puncta Volume</strong> An average number of voxels (volume picture elements) of the object times z,y, and z calibrations. - <strong>Puncta Number</strong> The average number of puncta counts within a given ROI (Region of Interest) - <strong>ROI Intensity</strong> The measures the overall pixel intensity over the Region of Interest - The results are summarized in MS Excel format.
</td>
<td><a href="https://github.com/DanHUMassMed">Dan Higgins</a>
</td></tr>

<tr valign="top">
<td><b>Qualitative Annotations</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Qualitative-Annotations/">https://sites.imagej.net/Qualitative-Annotations/</a>
</td>
<td>Plugins to describe images or rois with qualitative keywords (ground-truth classes, categories...) Selected keywords are reported in a result table and stored as roi properties.
</td>
<td><a href="https://imagej.net/people/LauLauThom">Laurent Thomas</a>
</td></tr>

<tr valign="top">
<td><b>Quantixed</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Quantixed/">https://sites.imagej.net/Quantixed/</a>
</td>
<td>ImageJ macros for making figures, annotating movies, blind analysis and more. Maintenance and issues <a href="https://github.com/quantixed/imagej-macros/">here</a>.
</td>
<td><a href="https://github.com/quantixed">Stephen Royle</a>
</td></tr>

<tr valign="top">
<td><b>QuickFigures</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/QuickFigures/">https://sites.imagej.net/QuickFigures/</a>
</td>
<td>ImageJ PlugIn for making publication quality figures from multichannel images. User friendly, fast, efficient and versatile. Automates dozens of steps. Comparable to ScientiFig, FigureJ and omero.figure. See youtube tutorial <a href="https://www.youtube.com/watch?v=uRGPBT1mhHg&list=PLM5I73cb55tCEjT7qdU7XDrAnSxg6ZNxp&index=1">here</a>. Also see publication for more details.
</td>
<td><a href="https://imagej.net/User:GregM">Greg M</a>
</td></tr>

<tr valign="top">
<td><b>QuimP</b>
</td>
<td><a rel="nofollow" class="external free" href="https://quimp.warwick.ac.uk/quimp-update-site/">https://quimp.warwick.ac.uk/quimp-update-site/</a>
</td>
<td><a href="https://warwick.ac.uk/quimp">QuimP</a> is software for tracking cellular shape changes and dynamic distributions of fluorescent reporters at the cell membrane.
</td>
<td>Piotr Baniukiewicz, Till Bretschneider
</td></tr>

<tr valign="top">
<td><b>Radial Symmetry</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/RadialSymmetry/">https://sites.imagej.net/RadialSymmetry/</a>
</td>
<td>A plugin for radial symmetry localization on smFISH (and other) images.
</td>
<td><a href="https://imagej.net/people/kephale">Kyle Harrington</a>
</td></tr>

<tr valign="top">
<td><b>RadialIntensityProfile</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/PTschaikner/">https://sites.imagej.net/PTschaikner/</a>
</td>
<td>RadialIntensityProfile is a simple macro to analyse the radial distribution of a fluorescent signal of interest in relation to a cell's nucleus.
</td>
<td><a href="https://imagej.net/User:PTschaikner">Philipp Tschaikner</a>
</td></tr>

<tr valign="top">
<td><b>ReadPlate</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ReadPlate/">https://sites.imagej.net/ReadPlate/</a>
</td>
<td>This macro measures absorbance values from an image of a multi-well plate. ReadPlate 3.0 version incorporates the capability of reading multi-well plates of 6 (3x2), 12 (4x3), 24 (6x4), 48 (8x6) and 96 (12x8) wells. An new user-friendly interface facilitates image analysis. This version allows new user-defined features to optimize blank correction. Multiple readings of the same plate has also considerably been facilitated, appending the full parameter file to the results and log file.
</td>
<td>José María Delfino (delfino@qb.ffyb.uba.ar)
</td></tr>

<tr valign="top">
<td><b>ResultsToExcel</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ResultsToExcel/">https://sites.imagej.net/ResultsToExcel/</a>
</td>
<td>This plugin extracts data from the Results Table and adds it to a new or existing .xlsx Excel file on the user's desktop. It works well with batching as data is not overwritten. A similar plugin exists by another developer but that one didn't work well for me. I did not consult their source code. <a href="https://www.youtube.com/watch?v=79rzUae-7N8/">YouTube tutorial</a>.
</td>
<td><a href="https://imagej.net/plugins/read-and-write-excel">Anthony Sinadinos</a>
</td></tr>

<tr valign="top">
<td><b>ROI 1-click Tools</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/RoiClicTool/">https://sites.imagej.net/RoiClicTool/</a>
</td>
<td>This update sites provides ROI tools similar to the built-in ImageJ ROI, but with additional functionalities. A single cick generates a predefined ROI, adds it to the RoiManager, run "Measure" for the outlined region, and go to the next slice for stacks. These options can be modified by double-clicking the ROI icon.
</td>
<td><a href="https://imagej.net/people/LauLauThom">Laurent Thomas</a>
</td></tr>

<tr valign="top">
<td><b>ROI-group Table</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/RoiGroupTable/">https://sites.imagej.net/RoiGroupTable/</a>
</td>
<td>This update sites provides a single helper plugin to set the category names associated to the ROI-group integer indexes via an intuitive table. The plugin also allows importing/exporting the category names mappings from/to a text file, for exchange between ImageJ/Fiji installations.
</td>
<td><a href="https://imagej.net/people/LauLauThom">Laurent Thomas</a>
</td></tr>

<tr valign="top">
<td><b>RT-Multiview-Deconvolution</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/RT-Multiview-Deconvolution/">https://sites.imagej.net/RT-Multiview-Deconvolution/</a>
</td>
<td>Plugins for plane-wise GPU-accelerated multi-view deconvolution of SPIM data.
</td>
<td><a href="https://imagej.net/people/bene51">Benjamin Schmid</a>
</td></tr>

<tr valign="top">
<td><b>SAMJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SAMJ/">https://sites.imagej.net/SAMJ/</a>
</td>
<td>SAMJ is a Fiji plugin for annotating microscopy images using various versions of the Segment Anything Model (SAM). See the <a href="https://github.com/segment-anything-models-java/">GitHub organization</a> for details.
</td>
<td><a href="https://imagej.net/people/carlosuc3m">Carlos Garcia</a>, <a href="mailto:daniel.sage@epfl.ch">Daniel Sage</a>, <a href="mailto:mamunozb@ing.uc3m.es">Arrate Muñoz-Barrutia</a>, <a href="https://imagej.net/people/xulman">Vladimír Ulman</a>, <a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>Sceptical Physiologist</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Scepticalphysiologist/">https://sites.imagej.net/Scepticalphysiologist/</a>
</td>
<td>FFT (spectra + filtering); ROI class management; Image Linker; Tight Montage; Ca2+ imaging tools; etc.
</td>
<td>Sean Parsons at <a href="http://scepticalphysiologist.com/code/code.html">scepticalphysiologist.com</a>
</td></tr>

<tr valign="top">
<td><b>SCF MPI CBG</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SCF-MPI-CBG/">https://sites.imagej.net/SCF-MPI-CBG/</a>
</td>
<td>Common image processing tools from the Scientific Computing Facility at MPI CBG Dresden
</td>
<td><a href="https://imagej.net/people/haesleinhuepf">Robert Haase</a>
</td></tr>

<tr valign="top">
<td><b>ScientiFig</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Aigouy/">https://sites.imagej.net/Aigouy/</a>
</td>
<td>ScientiFig is a tool to build publication-ready scientific figures similar to FigureJ from the IBMP-CNRS site. It must be used in combination with the 'Java-8' update site. ScientiFig is deprecated please use EZFig instead.
</td>
<td><a href="mailto:scientifig@gmail.com">Benoit Aigouy</a>
</td></tr>

<tr valign="top">
<td><b>Scientifig-deprecated</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SF-Deprecated-Legacy/">https://sites.imagej.net/SF-Deprecated-Legacy/</a>
</td>
<td>ScientiFig is a tool to build publication-ready scientific figures similar to FigureJ from the IBMP-CNRS site. This version of SF is deprecated. It must be used in combination with the 'Fiji-Legacy' update site (on old Fiji systems still using java 6). Note also that this version of SF is not compatible with 'Tissue Analyser'.
</td>
<td><a href="mailto:scientifig@gmail.com">Benoit Aigouy</a>
</td></tr>

<tr valign="top">
<td><b>Scijava Jupyter Kernel</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Scijava-jupyter-kernel/">https://sites.imagej.net/Scijava-jupyter-kernel/</a>
</td>
<td>A Java kernel for Jupyter. See source <a href="https://github.com/scijava/scijava-jupyter-kernel">on GitHub</a>.
</td>
<td><a href="https://imagej.net/people/hadim">Hadrien Mary</a>
</td></tr>

<tr valign="top">
<td><b>SciJava Ops</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/scijava-ops/">https://sites.imagej.net/scijava-ops/</a>
</td>
<td>A framework for extensible algorithm execution. REQUIRES JAVA 11+. See source <a href="https://github.com/scijava/scijava">on GitHub</a>.
</td>
<td><a href="https://imagej.net/people/gselzer">Gabriel Selzer</a>, <a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>, <a href="https://imagej.net/people/hinerm">Mark Hiner</a>
</td></tr>

<tr valign="top">
<td><b>sciview</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/sciview/">https://sites.imagej.net/sciview/</a>
</td>
<td>sciview 3D/VR/AR visualization package for ImageJ. See the documentation <a href="https://docs.scenery.graphics/sciview">here</a> and source <a href="https://github.com/scenerygraphics/sciview">on GitHub</a>.
</td>
<td><a href="https://imagej.net/people/kephale">Kyle Harrington</a>, <a href="https://imagej.net/people/skalarproduktraum">Ulrik Günther</a>
</td></tr>

<tr valign="top">
<td><b>Server</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Server/">https://sites.imagej.net/Server/</a>
</td>
<td>ImageJ-server update site. See source <a href="https://github.com/imagej/imagej-server">on GitHub</a>.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>SIMcheck</b>
</td>
<td><a rel="nofollow" class="external free" href="https://downloads.micron.ox.ac.uk/fiji_update/SIMcheck/">https://downloads.micron.ox.ac.uk/fiji_update/SIMcheck/</a>
</td>
<td><a href="https://www.micron.ox.ac.uk/microngroup/software/SIMCheck.php">SIMcheck</a>:
  a package of ImageJ tools for assessing the quality and reliability of
  Structured Illumination Microscopy (SIM) data.
</td>
<td><a href="https://www.micron.ox.ac.uk/home.php">Micron Oxford</a>
</td></tr>

<tr valign="top">
<td><b>SimonKlein</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SimonKlein/">https://sites.imagej.net/SimonKlein/</a>
</td>
<td>Provides the plugins <a href="https://github.com/kleinsimon/LinearDistance">LinearDistance</a> and <a href="https://github.com/kleinsimon/PointAnalysis">PointAnalysis</a>
</td>
<td><a href="https://imagej.net/User:SimonKlein">Simon Klein</a>
</td></tr>

<tr valign="top">
<td><b>SiMView</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SiMView/">https://sites.imagej.net/SiMView/</a>
</td>
<td>Tools developed by the Keller Lab at the Janelia Research Campus, most notably the <a href="https://bitbucket.org/fernandoamat/keller-lab-block-filetype">KLB</a> file format.
</td>
<td><a href="mailto:hoeckendorfb@janelia.hhmi.org">Burkhard Hoeckendorf</a>
</td></tr>

<tr valign="top">
<td><b>Slide Set</b>
</td>
<td><a rel="nofollow" class="external free" href="https://b.nanes.org/slideset/update/">https://b.nanes.org/slideset/update/</a>
</td>
<td><a href="https://b.nanes.org/slideset/">Slide Set</a>: Reproducible image analysis and batch processing.
</td>
<td><a href="https://imagej.net/people/bnanes">Benjamin Nanes</a>
</td></tr>

<tr valign="top">
<td><b>SlideBook</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SlideBook/">https://sites.imagej.net/SlideBook/</a>
</td>
<td>This Bio-Formats plugin enables the import of <a href="https://www.intelligent-imaging.com/slidebook">SlideBook</a> SLD files.
</td>
<td><a href="https://www.intelligent-imaging.com/contact">3i Support</a>
</td></tr>

<tr valign="top">
<td><b>SLIM Curve</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SLIM-Curve/">https://sites.imagej.net/SLIM-Curve/</a>
</td>
<td><a href="https://imagej.net/plugins/slim-curve">SLIM Curve</a>: a plugin for combined spectral-lifetime image analysis.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>SLIM-Notts</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SLIM-Notts/">https://sites.imagej.net/SLIM-Notts/</a>
</td>
<td>Plugins and macros from the School of Life Sciences Imaging (SLIM) at the University of Nottingham
</td>
<td>Chris Gell
</td></tr>

<tr valign="top">
<td><b>SpermQ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/SpermQ/">https://sites.imagej.net/SpermQ/</a>
</td>
<td>ImageJ plugins to analyze the flagellar beat and the swimming path of sperm from time-lapse 2D bright-field, dark-field, phase contrast, or fluorescence microscopy images. SpermQ can analyze images from head-tethered or free swimming sperm of different species. SpermQ can also analyze the beat of motile cilia if the analyzed cilium is clearly distinguishable from / not close to neighboring cilia. For more details, a link to the corresponding publication, and information on how to cite the plugins, visit the SpermQ github page https://github.com/hansenjn/SpermQ/
</td>
<td><a href="https://github.com/hansenjn/">Jan Hansen</a>
</td></tr>

<tr valign="top">
<td><b>Spindle3D</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Spindle3D/">https://sites.imagej.net/Spindle3D/</a>
</td>
<td>3D segmentation and morphometry of mitotic spindles
</td>
<td>Christian Tischer
</td></tr>

<tr valign="top">
<td><b>StarDist</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/StarDist/">https://sites.imagej.net/StarDist/</a>
</td>
<td>Object Detection with Star-convex Shapes
</td>
<td><a href="https://github.com/uschmidt83">Uwe Schmidt</a>, <a href="https://github.com/maweigert">Martin Weigert</a>
</td></tr>

<tr valign="top">
<td><b>Stereoscopic 3D Projection</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/JoeNapoli/">https://sites.imagej.net/JoeNapoli/</a>
</td>
<td>Allows users to create 3-D anaglyph and Google Cardboard compatible projections from image stacks.
</td>
<td>Joe Napoli, <a href="https://visikol.com">Visikol, Inc.</a>
</td></tr>

<tr valign="top">
<td><b>Stowers</b>
</td>
<td><a rel="nofollow" class="external free" href="https://research.stowers.org/imagejplugins/updates/">https://research.stowers.org/imagejplugins/updates/</a>
</td>
<td>The <a href="https://research.stowers.org/imagejplugins/">Stowers ImageJ Plugins</a>.
</td>
<td>Jay Unruh
</td></tr>

<tr valign="top">
<td><b>StuartLab</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/StuartLab/">https://sites.imagej.net/StuartLab/</a>
</td>
<td>Tools developed and used in the labs of Dr. Stuart and Dr. Necakov at Brock University.
</td>
<td><a href="https://imagej.net/orgs/stuart-lab">Andrew Valente</a>
</td></tr>

<tr valign="top">
<td><b>Tango</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Tango/">https://sites.imagej.net/Tango/</a>
</td>
<td>Tools for Analysis of Nuclear Genome Organization.
</td>
<td><a href="https://imagej.net/people/TcaManager">Jean Ollion, Julien Cochennec</a>
</td></tr>

<tr valign="top">
<td><b>TDS</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TDS/">https://sites.imagej.net/TDS/</a>
</td>
<td>Fiji-Tools-for-HCS is a collection of macros useful for high-content screening data developed by TDS members at MPI-CBG Dresden. <a href="https://github.com/stoeter/Fiji-Tools-for-HCS/wiki">Wiki</a>
</td>
<td><a href="mailto:stoeter@mpi-cbg.de">Martin Stoeter</a>
</td></tr>

<tr valign="top">
<td><b>TEM suite</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Makoten/">https://sites.imagej.net/Makoten/</a>
</td>
<td><a href="https://imagej.net/plugins/tem-suite">TEM suite</a> is a set of easy to use and useful plugins designed for analyzing and presenting TEM data and images. It includes: <a href="https://imagej.net/plugins/calc-dspace">calc dSpace</a>, which calculates the d-spacing from manual measurements made by the user in their electron diffraction patterns; <a href="https://imagej.net/plugins/get-dspace">get dSpace</a>, a more advanced d-spacing calculator; and <a href="https://imagej.net/plugins/pub-montage">pub Montage</a>, an image montaging plugin for journal article figures.
</td>
<td><a href="mailto:mark.koten@gmail.com">Mak Koten</a>
</td></tr>

<tr valign="top">
<td><b>TensorFlow</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TensorFlow/">https://sites.imagej.net/TensorFlow/</a>
</td>
<td><a href="https://imagej.net/software/tensorflow">TensorFlow</a> integration for ImageJ, including a microscope image focus quality classifier and a demo <a href="https://arxiv.org/abs/1512.00567">Inception</a> image classifier.
</td>
<td><a href="https://imagej.net/people/ctrueden">Curtis Rueden</a>
</td></tr>

<tr valign="top">
<td><b>Timebar</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Timebar/">https://sites.imagej.net/Timebar/</a>
</td>
<td>The <a href="https://github.com/remiberthoz/imagej-timebar">Timebar</a> plugin is intended for stacks with a temporal dimension (2D+T, 3D+T, etc.). Frames in the time series will be decorated with a timestamp and, optionally, a progressing bar.
</td>
<td><a href="mailto:remi.berthoz+imagej@gmail.com">Rémi Berthoz</a>
</td></tr>

<tr valign="top">
<td><b>Tissue Analyzer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TA/">https://sites.imagej.net/TA/</a>
</td>
<td>Tissue Analyzer is a free tool to segment epithelial tissues.
</td>
<td><a href="mailto:tissue.analyser@gmail.com">Benoit Aigouy</a>
</td></tr>

<tr valign="top">
<td><b>TomancakLab</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TomancakLab/">https://sites.imagej.net/TomancakLab/</a>
</td>
<td>A couple of Fiji scripts to ease daily evo-devo image analysis life, from Tomancak lab.
</td>
<td><a href="mailto:ulman@mpi-cbg.de">Vladimír Ulman</a>, <a href="https://imagej.net/people/maarzt">Matthias Arzt</a>
</td></tr>

<tr valign="top">
<td><b>Tr2d</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Tr2d/">https://sites.imagej.net/Tr2d/</a>
</td>
<td>Tracking in 2d.
</td>
<td><a href="mailto:jug@mpi-cbg.de">Florian Jug</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Cellpose</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Cellpose">https://sites.imagej.net/TrackMate-Cellpose</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-cellpose">TrackMate-Cellpose</a>, a detector module for TrackMate that relies on <a href="https://cellpose.readthedocs.io/en/latest/">cellpose</a>.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-ExTrack</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-ExTrack">https://sites.imagej.net/TrackMate-ExTrack</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-extrack">TrackMate-ExTrack</a>, a track analysis module for TrackMate, that is a port of the ExTrack analysis tool written in Python. ExTrack can reveal diffusion and binding kinetics in live cells and beyond. It is made to be robust against noise, notably if displacements are small and state transitions are fast.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Helper</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Helper">https://sites.imagej.net/TrackMate-Helper</a>
</td>
<td>The TrackMate-Helper is a plugin made to run parameter sweeps on an image to track. It allows a user to configure a very large batch of tracking parameters to test with TrackMate, and will run them. The Cell-Tracking-Challenge metrics are calculated for each parameter set tested, which allows to find the best tracking configuration for a dataset with respect to a specific tracking metric. Check <a href="https://imagej.net/plugins/trackmate/extensions/trackmate-helper">here</a> for documentation.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Ilastik</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Ilastik">https://sites.imagej.net/TrackMate-Ilastik</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-ilastik">TrackMate-Ilastik</a>, a detector module for TrackMate that relies on Ilastik to segment objects in 2D or 3D.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Lacss</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Lacss">https://sites.imagej.net/TrackMate-Lacss</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-lacss">TrackMate-Lacss</a>, a pretrained deep-learning library called <a href="https://github.com/jiyuuchc/lacss">Lacss</a> to detect and segment cells serving as a detector to the popular cell/object tracker plugin <a href="https://imagej.net/plugins/trackmate/index">TrackMate</a>. This plugin REQUIRES a python enviorment setup. See the lacss github or <a href="https://jiyuuchc.github.io/lacss/install/">documentation</a>. This plugin is overall is thin Java Wrapper around the Lacss python module, which runs as a seperate process. Java communicates with the python process via anonymous pipe - sending messages encoded in protobuf.
</td>
<td><a href="https://github.com/jiyuuchc">Ji Yu</a>, <a href="https://github.com/Nick-Kuang">Nicholas Kuang</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-MorphoLibJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-MorphoLibJ">https://sites.imagej.net/TrackMate-MorphoLibJ</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-morpholibj">TrackMate-MorphoLibJ</a>, a detector module for TrackMate that relies on the <a href="https://imagej.net/plugins/morphological-segmentation">Morphological Segmentation</a> plugin shipped with the <a href="https://imagej.net/plugins/morpholibj">MorphoLibJ</a> library.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Oneat</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Oneat/">https://sites.imagej.net/TrackMate-Oneat/</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-oneat">TrackMate-Oneat</a>, a track corrector module for TrackMate that relies on <a href="https://pypi.org/project/oneat/">oneat</a>.
</td>
<td><a href="https://imagej.net/people/kapoorlab">Varun Kapoor</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Pairing</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Pairing">https://sites.imagej.net/TrackMate-Pairing</a>
</td>
<td>Contains a tool that allows for pairing together two TrackMate files, based on several distance metrics. It builds pairs taken from the tracks of one file and the tracks of a second file. This is used mainly to study how the distance of objects detected in two different channels evolve over time.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-StarDist</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-StarDist">https://sites.imagej.net/TrackMate-StarDist</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-stardist">TrackMate-StarDist</a>, a detector module for TrackMate that relies on <a href="https://imagej.net/plugins/stardist">StarDist</a> to segment objects in 2D.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMate-Weka</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMate-Weka">https://sites.imagej.net/TrackMate-Weka</a>
</td>
<td>This update site provides <a href="https://imagej.net/plugins/trackmate/trackmate-weka">TrackMate-Weka</a>, a detector module for TrackMate that relies on the <a href="https://imagej.net/plugins/tws/index">Trainable Weka Segmentation plugin</a> to segment objects in 2D or 3D.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TrackMateCSVImporter</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TrackMateCSVImporter/">https://sites.imagej.net/TrackMateCSVImporter/</a>
</td>
<td>The <a href="https://imagej.net/plugins/trackmate/csv-importer">TrackMateCSVImporter</a> is a Fiji plugin that can import CSV files into TrackMate. It can create TrackMate files out of a csv file and an image. A script also permits its use in headless mode.
</td>
<td><a href="https://imagej.net/people/tinevez">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>TraJClassifier</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TraJClassifier/">https://sites.imagej.net/TraJClassifier/</a>
</td>
<td>The <a href="https://imagej.net/plugins/trajclassifier">TraJClassifier</a> loads trajectories from TrackMate, characterize them using TraJ and classificate them into normal diffusion, subdiffusion, confined diffusion and directed/active motion by a random forest approach.
</td>
<td><a href="https://imagej.net/people/thorstenwagner">Thorsten Wagner</a>
</td></tr>

<tr valign="top">
<td><b>TreeJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TreeJ/">https://sites.imagej.net/TreeJ/</a>
</td>
<td><a href="https://imagej.net/plugins/treej">TreeJ</a> allows interactive cell lineage reconstruction and annotation from a static labeled image.
</td>
<td><a href="https://www.slcu.cam.ac.uk/staff/elise-laruelle">Elise Laruelle</a>
</td></tr>

<tr valign="top">
<td><b>TWOMBLI</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/TWOMBLI/">https://sites.imagej.net/TWOMBLI/</a>
</td>
<td>Plugin implementation of <a href="https://dx.doi.org/10.26508/lsa.202000880">TWOMBLI</a>, for the analysis of network structures such as extra-cellular matrix, vascular or cytoskeletal components.
</td>
<td><a href="mailto:david.barry@crick.ac.uk">David Barry</a>
</td></tr>

<tr valign="top">
<td><b>U-Net Segmentation</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Falk/">https://sites.imagej.net/Falk/</a>
</td>
<td>The U-Net segmentation plugin interfaces caffe on a remote Linux host to segment images using artificial neural networks. The Plugin requires a special extended version of caffe that is available via the <a href="https://lmb.informatik.uni-freiburg.de/resources/opensource/unet">U-Net Project Page</a>. This plugin is maintained on <a href="https://github.com/lmb-freiburg/Unet-Segmentation">GitHub</a>.
</td>
<td><a href="mailto:falk@informatik.uni-freiburg.de">Thorsten Falk</a>
</td></tr>

<tr valign="top">
<td><b>UCB Vision Sciences</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Llamero/">https://sites.imagej.net/Llamero/</a>
</td>
<td>A suite of tools developed as part of the University of California, Berkeley Vision Sciences core grant NIH P30EY003176, including a Hough Circle Transform and a Quiver Plot tool.
</td>
<td><a href="mailto:benjamin.smith@berkeley.edu">Ben Smith</a>
</td></tr>

<tr valign="top">
<td><b>Ultrack</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ultrack/">https://sites.imagej.net/ultrack/</a>
</td>
<td><a href="https://imagej.net/plugins/ultrack">Ultrack</a> is a large-scale cell tracking under segmentation uncertainty. Capable of managing millions of segmentation instances, it provides competitive results with or without deep learning. The method optimizes tracking by using a hierarchy of segmentation hypotheses to select segments across frames.
</td>
<td><a href="mailto:ilan.silva@czbiohub.org">Ilan Theodoro</a>, <a href="mailto:jordao.bragantini@czbiohub.org">Jordao Bragantini</a>
</td></tr>

<tr valign="top">
<td><b>Vale lab plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Nico/">https://sites.imagej.net/Nico/</a>
</td>
<td>Plugins useful to the Vale lab: Saim, spot intensity analysis, spot counter.
</td>
<td><a href="https://imagej.net/people/nicost">Nico Stuurman</a>
</td></tr>

<tr valign="top">
<td><b>Vanderbilt Biophotonics Center</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Wilson.adams/">https://sites.imagej.net/Wilson.adams/</a>
</td>
<td>Customized GUI and image analysis scripts designed for local use by the <a href="https://www.vanderbilt.edu/vbc/">Vanderbilt Biophotonics Center</a> and collaborators. Avoid following if you have a personally customized StartupMacros.ijm file.
</td>
<td><a href="mailto:wilson.adams@vanderbilt.edu">Wilson R Adams</a> Vanderbilt Biophotonics Center, Nashville, TN, USA
</td></tr>

<tr valign="top">
<td><b>VCell Simulation Results Viewer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/VCell-Simulations-Result-Viewer/">https://sites.imagej.net/VCell-Simulations-Result-Viewer/</a>
</td>
<td>Plugin for accesing spatial simulation results exported in the N5 file format from VCell.
</td>
<td><a href="mailto:evalencia@uchc.edu">Ezequiel Valencia</a>, <a href="mailto:schaff.jim@gmail.com">Jim Schaff</a>
</td></tr>

<tr valign="top">
<td><b>VerciniAnalysisJ</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/VerciniAnalysisJ/">https://sites.imagej.net/VerciniAnalysisJ/</a>
</td>
<td>A set of ImageJ scripts for analysis of VerCINI (Vertical Cell Imaging by Nanostructured Immobilization) microscopy data of vertically oriented bacteria. See https://github.com/HoldenLab/VerciniAnalysisJ for documentation.
</td>
<td><a href="mailto:seamus.holden@ncl.ac.uk">Seamus Holden</a> Newcastle University, UK
</td></tr>

<tr valign="top">
<td><b>VersiLab plugins</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Sweeneyian/">https://sites.imagej.net/Sweeneyian/</a>
</td>
<td>VersiLab_Plugin, aiming to do real-time image acquisition and processing for Optical Endo-Microscopy (OEM), development phase, code available <a href="https://github.com/Sweeneyian/VersiLab_Plugin">on GitHub</a>.
</td>
<td><a href="https://imagej.net/people/sweeneyian">Ian Sweeney</a>
</td></tr>

<tr valign="top">
<td><b>Virtual-Orientation-Tools-VOTj</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Virtual-Orientation-Tool-VOTj/">https://sites.imagej.net/Virtual-Orientation-Tool-VOTj/</a>
</td>
<td><a href="https://github.com/sankeert1999/Virtual-Orientation-Tools-VOTj">Virtual-Orientation-Tools-VOTj</a> is a set of Fiji plugins designed to center and align objects of interest both horizontally and vertically from their base orientation. The program is available through a Fiji plugin.
</td>
<td><a href="mailto:sankeert1999@gmail.com">Sankeert Satheesan</a>
</td></tr>

<tr valign="top">
<td><b>Visualization Toolset</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/VisualizationToolset/">https://sites.imagej.net/VisualizationToolset/</a>
</td>
<td>A <a href="https://kwolby.notion.site/kwolby/Visualization-Toolset-4e0a4f2834494ba5a4865a99c6913323">macro Toolset</a> helping daily handling and visualization of multichannel images and stacks. Feedback always welcome!
</td>
<td><a href="mailto:kevinterretaz@gmail.com">Kévin Terretaz</a>
</td></tr>

<tr valign="top">
<td><b>Void Whizzard</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/VezinaLab/">https://sites.imagej.net/VezinaLab/</a>
</td>
<td><a href="https://imagej.net/plugins/void-whizzard">Void Whizzard</a> is a tool for analyzing void spot assay (VSA) images.
</td>
<td><a href="mailto:soakes@wisc.edu">Royal Oakes</a>
</td></tr>

<tr valign="top">
<td><b>Volumetric Tissue Exploration and Analysis</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ICBM-IUPUI/">https://sites.imagej.net/ICBM-IUPUI/</a>
</td>
<td><a href="https://imagej.net/plugins/vtea">VTEA</a> is an interactive tool for 3D tissue cytometry built on ImageJ.
</td>
<td><a href="mailto:winfrees@iu.edu">Seth Winfree</a>
</td></tr>

<tr valign="top">
<td><b>WhiskerTracking</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Tarokiritani/">https://sites.imagej.net/Tarokiritani/</a>
</td>
<td><a href="https://github.com/tarokiritani/WhiskerTracking">WhisketTracking</a> is a set of tools for tracking rodent whisker.
</td>
<td><a href="mailto:taro.kiritani@epfl.ch">Taro Kiritani</a>
</td></tr>

<tr valign="top">
<td><b>WormBox</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/WormBox/">https://sites.imagej.net/WormBox/</a>
</td>
<td><a href="https://github.com/bruvellu/WormBox">WormBox</a> is a plugin for measuring linear distances and counting structures based on landmarks.
</td>
<td><a href="https://brunovellutini.com/">Bruno C. Vellutini</a>, <a href="http://lhe.ib.usp.br/lhe/">Fernando P. L. Marques</a>
</td></tr>

<tr valign="top">
<td><b>WormSizer</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Bradtmoore/">https://sites.imagej.net/Bradtmoore/</a>
</td>
<td>The update site for <a href="https://github.com/bradtmoore/wormsizer">WormSizer</a>, a tool for measuring volume, length, and surface area of nematodes off of a dissection microscope.
</td>
<td><a href="mailto:brad.t.moore@duke.edu">Brad T. Moore</a>
</td></tr>

<tr valign="top">
<td><b>XitoSBML</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/XitoSBML/">https://sites.imagej.net/XitoSBML/</a>
</td>
<td><a href="https://github.com/spatialsimulator/XitoSBML">XitoSBML</a> is an ImageJ plugin which creates a <a href="http://sbml.org/Documents/Specifications/SBML_Level_3/Packages/spatial">Spatial SBML</a> model from segmented images. XitoSBML is not just a converter, but also a spatial model editor so that users can add molecules(species), reactions and advection/diffusion coefficients to the converted Spatial SBML model.
</td>
<td><a href="mailto:spatial@fun.bio.keio.ac.jp">Kaito Ii</a>, <a href="mailto:spatial@fun.bio.keio.ac.jp">Akira Funahashi</a>
</td></tr>

<tr valign="top">
<td><b>Xlib</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Xlib/">https://sites.imagej.net/Xlib/</a>
</td>
<td><a href="https://imagej.net/plugins/xlib">Xlib</a> is a set of prospective ImageJ plugins: automated imaging tools for filtering, data reconstruction, quantitative data evaluation and data import, tools for interactive segmentation, visualization and management of image data.
</td>
<td><a href="mailto:beat.muench@empa.ch">Beat Münch</a>
</td></tr>

<tr valign="top">
<td><b>ZedMate Suite</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Ayakimovich/">https://sites.imagej.net/Ayakimovich/</a>
</td>
<td><a href="https://imagej.net/plugins/zedmate">ZedMate</a> is a TrackMate-based plugin for 3D particles detection and measurement.
</td>
<td><a href="mailto:a.yakimovich@ucl.ac.uk">Artur Yakimovich</a>
</td></tr>

<tr valign="top">
<td><b>Zeiss Quick Start Reader</b>
</td>
<td><a rel="nofollow" class="external free" href="https://biop.epfl.ch/Fiji-CZI/">https://biop.epfl.ch/Fiji-CZI/</a>
</td>
<td>An alternative Bio-Formats Zeiss reader for big files. Documentation: <a href="https://imagej.net/plugins/zeiss-quick-start-reader">Zeiss Quick Start Reader</a>.
</td>
<td><a href="mailto:nicolas.chiaruttini@epfl.ch">Nicolas Chiaruttini</a>
</td></tr>

<tr valign="top">
<td><b>Zellige</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Zellige/">https://sites.imagej.net/Zellige/</a>
</td>
<td><a href="https://imagej.net/plugins/zellige">Zellige</a> is a software tool allowing the automated extraction of a non-prescribed number of various surfaces from a 3D image acquired by fluorescence microscopy. The program is available through a Fiji plugin.
</td>
<td><a href="mailto:ctrebeau@pasteur.fr">Céline Trébeau</a>, <a href="mailto:tinevez@pasteur.fr">Jean-Yves Tinevez</a>
</td></tr>

<tr valign="top">
<td><b>ZFBONE</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/Zfmacros/">https://sites.imagej.net/Zfmacros/</a>
</td>
<td><a href="https://sites.imagej.net/Zfmacros">ZFBONE</a> is a toolset that provides tools needed for a rapid, reliable and semi-automatized morphometric analysis of zebrafish bone structures such as operculum, caudal fin and scales.
</td>
<td><a href="mailto:mtarasco@ualg.pt">Marco Tarasco</a>
</td></tr>

<tr valign="top">
<td><b>Zoom-in-movie</b>
</td>
<td><a rel="nofollow" class="external free" href="https://sites.imagej.net/ZOOM-in-movie/">https://sites.imagej.net/ZOOM-in-movie/</a>
</td>
<td><a href="https://imagej.net/FIXME">Zoom-in-movie</a> is a simple tool for creating zoom-in movies with logarithmic approach and dynamic scalebar. Simply select ROI on your image and select Plugins/Zoom-in movie to run the plugin. Thanks to Eugene Katrukha for the initial implementation.
</td>
<td><a href="mailto:aaristov@pasteur.fr">Andrey Aristov</a>
</td></tr>

</table>
				</div>
			</div>

			<div id="footer">
				<p>This page was last modified on 11 December 2024, at 16:37.</p>
			</div>
		</div>

		</body>
		</html>
		